diff -Naur a/Zorro.Core.Runtime/Zorro.Core.Runtime.csproj Out/Zorro.Core.Runtime/Zorro.Core.Runtime.csproj
--- a/Zorro.Core.Runtime/Zorro.Core.Runtime.csproj	2024-05-13 04:21:40.165811904 +0000
+++ Out/Zorro.Core.Runtime/Zorro.Core.Runtime.csproj	2024-05-13 04:22:14.613811676 +0000
@@ -8,8 +8,6 @@
     <LangVersion>11.0</LangVersion>
     <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
   </PropertyGroup>
-  <PropertyGroup />
-  <ItemGroup />
   <ItemGroup>
     <Reference Include="Unity.Mathematics">
       <HintPath>../../Temp/Assemblies/Unity.Mathematics.dll</HintPath>
diff -Naur a/Zorro.PhotonUtility/Zorro.PhotonUtility.csproj Out/Zorro.PhotonUtility/Zorro.PhotonUtility.csproj
--- a/Zorro.PhotonUtility/Zorro.PhotonUtility.csproj	2024-05-13 04:21:40.209811904 +0000
+++ Out/Zorro.PhotonUtility/Zorro.PhotonUtility.csproj	2024-05-13 04:22:39.621811510 +0000
@@ -21,10 +21,12 @@
       <HintPath>../../Temp/Assemblies/Zorro.Core.Runtime.dll</HintPath>
     </Reference>
   </ItemGroup>
-    <ItemGroup><Reference Include="UnityEngine">
-      <HintPath>../../Temp/Assemblies/UnityEngine.dll</HintPath>
+  <ItemGroup>
+    <Reference Include="UnityEngine">
+      <HintPath>../../Temp/Assemblies/UnityEngine.dll</HintPath>
     </Reference>
     <Reference Include="UnityEngine.CoreModule">
-      <HintPath>../../Temp/Assemblies/UnityEngine.CoreModule.dll</HintPath>
-    </Reference></ItemGroup>
+      <HintPath>../../Temp/Assemblies/UnityEngine.CoreModule.dll</HintPath>
+    </Reference>
+  </ItemGroup>
 </Project>
\ No newline at end of file
diff -Naur a/Zorro.Settings.Runtime/Zorro.Settings.Runtime.csproj Out/Zorro.Settings.Runtime/Zorro.Settings.Runtime.csproj
--- a/Zorro.Settings.Runtime/Zorro.Settings.Runtime.csproj	2024-05-13 04:21:40.205811904 +0000
+++ Out/Zorro.Settings.Runtime/Zorro.Settings.Runtime.csproj	2024-05-13 04:22:58.949811381 +0000
@@ -30,7 +30,9 @@
       <HintPath>../../Temp/Assemblies/Zorro.Core.Runtime.dll</HintPath>
     </Reference>
   </ItemGroup>
-    <ItemGroup><Reference Include="Sirenix.Serialization">
-      <HintPath>../../Temp/Assemblies/Sirenix.Serialization.dll</HintPath>
-    </Reference></ItemGroup>
+  <ItemGroup>
+    <Reference Include="Sirenix.Serialization">
+      <HintPath>../../Temp/Assemblies/Sirenix.Serialization.dll</HintPath>
+    </Reference>
+  </ItemGroup>
 </Project>
\ No newline at end of file
