diff -Naur a/Assembly-CSharp/fix/ModifierID.cs out/Assembly-CSharp/fix/ModifierID.cs
--- a/Assembly-CSharp/fix/ModifierID.cs	1970-01-01 00:00:00.000000000 +0000
+++ out/Assembly-CSharp/fix/ModifierID.cs	2024-05-12 23:02:35.087830112 +0000
@@ -0,0 +1,7 @@
+using System;
+
+public class ModifierID: Attribute {
+    public ModifierID(string name) {
+        // NOOP
+    }
+}
\ No newline at end of file
diff -Naur a/Assembly-CSharp/fix/ProceduralImage.cs out/Assembly-CSharp/fix/ProceduralImage.cs
--- a/Assembly-CSharp/fix/ProceduralImage.cs	1970-01-01 00:00:00.000000000 +0000
+++ out/Assembly-CSharp/fix/ProceduralImage.cs	2024-05-12 23:04:27.847829130 +0000
@@ -0,0 +1,3 @@
+namespace UnityEngine.UI.ProceduralImage;
+
+public class ProceduralImage {}
diff -Naur a/Assembly-CSharp/fix/ProceduralImageModifier.cs out/Assembly-CSharp/fix/ProceduralImageModifier.cs
--- a/Assembly-CSharp/fix/ProceduralImageModifier.cs	1970-01-01 00:00:00.000000000 +0000
+++ out/Assembly-CSharp/fix/ProceduralImageModifier.cs	2024-05-12 23:03:29.159829641 +0000
@@ -0,0 +1,5 @@
+using UnityEngine;
+
+public abstract class ProceduralImageModifier {
+    public abstract Vector4 CalculateRadius(Rect imageRect);
+}
