diff -Naur a/Assembly-CSharp/CurvedUI/CurvedUIHandSwitcher.cs out/Assembly-CSharp/CurvedUI/CurvedUIHandSwitcher.cs
--- a/Assembly-CSharp/CurvedUI/CurvedUIHandSwitcher.cs	2024-05-12 23:22:41.319819608 +0000
+++ out/Assembly-CSharp/CurvedUI/CurvedUIHandSwitcher.cs	2024-05-12 23:21:52.755820031 +0000
@@ -2,6 +2,7 @@
 using System.Runtime.CompilerServices;
 using BepInEx.AssemblyPublicizer;
 using UnityEngine;
+using DefaultNamespace;
 
 namespace CurvedUI;
 
diff -Naur a/Assembly-CSharp/CurvedUI/CurvedUISettings.cs out/Assembly-CSharp/CurvedUI/CurvedUISettings.cs
--- a/Assembly-CSharp/CurvedUI/CurvedUISettings.cs	2024-05-12 23:22:36.583819649 +0000
+++ out/Assembly-CSharp/CurvedUI/CurvedUISettings.cs	2024-05-12 23:21:52.755820031 +0000
@@ -4,6 +4,7 @@
 using System.Runtime.CompilerServices;
 using BepInEx.AssemblyPublicizer;
 using UnityEngine;
+using DefaultNamespace;
 
 namespace CurvedUI;
 
