diff -Naur a/Assembly-CSharp/AnimatedItem.cs Out/Assembly-CSharp/AnimatedItem.cs
--- a/Assembly-CSharp/AnimatedItem.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/AnimatedItem.cs	2024-05-14 01:05:35.620634550 +0000
@@ -95,14 +95,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/AnimatedObjectTrigger.cs Out/Assembly-CSharp/AnimatedObjectTrigger.cs
--- a/Assembly-CSharp/AnimatedObjectTrigger.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/AnimatedObjectTrigger.cs	2024-05-14 01:05:35.500634546 +0000
@@ -244,7 +244,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -287,7 +287,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/Anomaly.cs Out/Assembly-CSharp/Anomaly.cs
--- a/Assembly-CSharp/Anomaly.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/Anomaly.cs	2024-05-14 01:05:35.548634547 +0000
@@ -71,7 +71,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -93,7 +93,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/AudioReverbTrigger.cs Out/Assembly-CSharp/AudioReverbTrigger.cs
--- a/Assembly-CSharp/AudioReverbTrigger.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/AudioReverbTrigger.cs	2024-05-14 01:05:35.568634548 +0000
@@ -163,14 +163,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/AutoParentToShip.cs Out/Assembly-CSharp/AutoParentToShip.cs
--- a/Assembly-CSharp/AutoParentToShip.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/AutoParentToShip.cs	2024-05-14 01:05:35.948634560 +0000
@@ -135,14 +135,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/BaboonBirdAI.cs Out/Assembly-CSharp/BaboonBirdAI.cs
--- a/Assembly-CSharp/BaboonBirdAI.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/BaboonBirdAI.cs	2024-05-14 01:05:35.568634548 +0000
@@ -780,7 +780,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -1005,7 +1005,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/BinocularsItem.cs Out/Assembly-CSharp/BinocularsItem.cs
--- a/Assembly-CSharp/BinocularsItem.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/BinocularsItem.cs	2024-05-14 01:05:35.948634560 +0000
@@ -14,14 +14,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/BlobAI.cs Out/Assembly-CSharp/BlobAI.cs
--- a/Assembly-CSharp/BlobAI.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/BlobAI.cs	2024-05-14 01:05:35.568634548 +0000
@@ -274,7 +274,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -303,7 +303,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/BoomboxItem.cs Out/Assembly-CSharp/BoomboxItem.cs
--- a/Assembly-CSharp/BoomboxItem.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/BoomboxItem.cs	2024-05-14 01:05:35.568634548 +0000
@@ -151,14 +151,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/BreakerBox.cs Out/Assembly-CSharp/BreakerBox.cs
--- a/Assembly-CSharp/BreakerBox.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/BreakerBox.cs	2024-05-14 01:05:35.780634555 +0000
@@ -99,14 +99,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/BridgeTrigger.cs Out/Assembly-CSharp/BridgeTrigger.cs
--- a/Assembly-CSharp/BridgeTrigger.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/BridgeTrigger.cs	2024-05-14 01:05:35.620634550 +0000
@@ -122,7 +122,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -151,7 +151,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/BridgeTriggerType2.cs Out/Assembly-CSharp/BridgeTriggerType2.cs
--- a/Assembly-CSharp/BridgeTriggerType2.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/BridgeTriggerType2.cs	2024-05-14 01:05:35.948634560 +0000
@@ -38,7 +38,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -60,7 +60,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ButlerBeesEnemyAI.cs Out/Assembly-CSharp/ButlerBeesEnemyAI.cs
--- a/Assembly-CSharp/ButlerBeesEnemyAI.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/ButlerBeesEnemyAI.cs	2024-05-14 01:05:35.500634546 +0000
@@ -46,14 +46,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ButlerEnemyAI.cs Out/Assembly-CSharp/ButlerEnemyAI.cs
--- a/Assembly-CSharp/ButlerEnemyAI.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/ButlerEnemyAI.cs	2024-05-14 01:05:35.692634552 +0000
@@ -660,7 +660,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -843,7 +843,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/CentipedeAI.cs Out/Assembly-CSharp/CentipedeAI.cs
--- a/Assembly-CSharp/CentipedeAI.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/CentipedeAI.cs	2024-05-14 01:05:35.620634550 +0000
@@ -618,7 +618,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -696,7 +696,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ClipboardItem.cs Out/Assembly-CSharp/ClipboardItem.cs
--- a/Assembly-CSharp/ClipboardItem.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/ClipboardItem.cs	2024-05-14 01:05:35.912634559 +0000
@@ -47,14 +47,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/CrawlerAI.cs Out/Assembly-CSharp/CrawlerAI.cs
--- a/Assembly-CSharp/CrawlerAI.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/CrawlerAI.cs	2024-05-14 01:05:35.692634552 +0000
@@ -320,7 +320,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -405,7 +405,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/DepositItemsDesk.cs Out/Assembly-CSharp/DepositItemsDesk.cs
--- a/Assembly-CSharp/DepositItemsDesk.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/DepositItemsDesk.cs	2024-05-14 01:05:35.780634555 +0000
@@ -652,7 +652,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -765,7 +765,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/DigitalRuby/ThunderAndLightning/SingleLineClampAttribute.cs Out/Assembly-CSharp/DigitalRuby/ThunderAndLightning/SingleLineClampAttribute.cs
--- a/Assembly-CSharp/DigitalRuby/ThunderAndLightning/SingleLineClampAttribute.cs	2024-05-14 01:01:33.180627075 +0000
+++ Out/Assembly-CSharp/DigitalRuby/ThunderAndLightning/SingleLineClampAttribute.cs	2024-05-14 01:07:53.624638805 +0000
@@ -41,7 +41,7 @@
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
-	public SingleLineClampAttribute(string tooltip, double minValue, double maxValue)
+	public SingleLineClampAttribute(string tooltip, double minValue, double maxValue): base(tooltip)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/DocileLocustBeesAI.cs Out/Assembly-CSharp/DocileLocustBeesAI.cs
--- a/Assembly-CSharp/DocileLocustBeesAI.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/DocileLocustBeesAI.cs	2024-05-14 01:05:35.692634552 +0000
@@ -120,14 +120,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/DoorLock.cs Out/Assembly-CSharp/DoorLock.cs
--- a/Assembly-CSharp/DoorLock.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/DoorLock.cs	2024-05-14 01:05:35.780634555 +0000
@@ -194,7 +194,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -265,7 +265,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/DoublewingAI.cs Out/Assembly-CSharp/DoublewingAI.cs
--- a/Assembly-CSharp/DoublewingAI.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/DoublewingAI.cs	2024-05-14 01:05:35.912634559 +0000
@@ -293,7 +293,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -329,7 +329,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/DressGirlAI.cs Out/Assembly-CSharp/DressGirlAI.cs
--- a/Assembly-CSharp/DressGirlAI.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/DressGirlAI.cs	2024-05-14 01:05:35.500634546 +0000
@@ -362,7 +362,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -412,7 +412,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/EnemyAI.cs Out/Assembly-CSharp/EnemyAI.cs
--- a/Assembly-CSharp/EnemyAI.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/EnemyAI.cs	2024-05-14 01:05:35.948634560 +0000
@@ -758,7 +758,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -857,7 +857,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/EnemyVent.cs Out/Assembly-CSharp/EnemyVent.cs
--- a/Assembly-CSharp/EnemyVent.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/EnemyVent.cs	2024-05-14 01:05:35.692634552 +0000
@@ -79,7 +79,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -108,7 +108,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/EntranceTeleport.cs Out/Assembly-CSharp/EntranceTeleport.cs
--- a/Assembly-CSharp/EntranceTeleport.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/EntranceTeleport.cs	2024-05-14 01:05:35.620634550 +0000
@@ -188,7 +188,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -217,7 +217,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/EventWhenDroppedItem.cs Out/Assembly-CSharp/EventWhenDroppedItem.cs
--- a/Assembly-CSharp/EventWhenDroppedItem.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/EventWhenDroppedItem.cs	2024-05-14 01:05:35.692634552 +0000
@@ -48,14 +48,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ExtensionLadderItem.cs Out/Assembly-CSharp/ExtensionLadderItem.cs
--- a/Assembly-CSharp/ExtensionLadderItem.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/ExtensionLadderItem.cs	2024-05-14 01:05:35.912634559 +0000
@@ -215,14 +215,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/FlashlightItem.cs Out/Assembly-CSharp/FlashlightItem.cs
--- a/Assembly-CSharp/FlashlightItem.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/FlashlightItem.cs	2024-05-14 01:05:35.500634546 +0000
@@ -114,7 +114,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -143,7 +143,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/FlowerSnakeEnemy.cs Out/Assembly-CSharp/FlowerSnakeEnemy.cs
--- a/Assembly-CSharp/FlowerSnakeEnemy.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/FlowerSnakeEnemy.cs	2024-05-14 01:05:35.548634547 +0000
@@ -424,7 +424,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -523,7 +523,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/FlowermanAI.cs Out/Assembly-CSharp/FlowermanAI.cs
--- a/Assembly-CSharp/FlowermanAI.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/FlowermanAI.cs	2024-05-14 01:05:35.948634560 +0000
@@ -363,7 +363,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -441,7 +441,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ForestGiantAI.cs Out/Assembly-CSharp/ForestGiantAI.cs
--- a/Assembly-CSharp/ForestGiantAI.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/ForestGiantAI.cs	2024-05-14 01:05:35.780634555 +0000
@@ -460,7 +460,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -524,7 +524,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/GameNetcodeStuff/PlayerControllerB.cs Out/Assembly-CSharp/GameNetcodeStuff/PlayerControllerB.cs
--- a/Assembly-CSharp/GameNetcodeStuff/PlayerControllerB.cs	2024-05-14 01:01:33.180627075 +0000
+++ Out/Assembly-CSharp/GameNetcodeStuff/PlayerControllerB.cs	2024-05-14 01:06:16.696635816 +0000
@@ -12,6 +12,7 @@
 using UnityEngine.Animations.Rigging;
 using UnityEngine.InputSystem;
 using UnityEngine.UI;
+using DefaultNamespace;
 
 namespace GameNetcodeStuff;
 
@@ -2562,7 +2563,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -2976,7 +2977,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/GiftBoxItem.cs Out/Assembly-CSharp/GiftBoxItem.cs
--- a/Assembly-CSharp/GiftBoxItem.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/GiftBoxItem.cs	2024-05-14 01:05:35.500634546 +0000
@@ -254,7 +254,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -290,7 +290,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/GlobalEffects.cs Out/Assembly-CSharp/GlobalEffects.cs
--- a/Assembly-CSharp/GlobalEffects.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/GlobalEffects.cs	2024-05-14 01:05:35.488634546 +0000
@@ -129,7 +129,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -186,7 +186,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/GrabbableObject.cs Out/Assembly-CSharp/GrabbableObject.cs
--- a/Assembly-CSharp/GrabbableObject.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/GrabbableObject.cs	2024-05-14 01:05:35.688634552 +0000
@@ -448,7 +448,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -582,7 +582,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/GunAmmo.cs Out/Assembly-CSharp/GunAmmo.cs
--- a/Assembly-CSharp/GunAmmo.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/GunAmmo.cs	2024-05-14 01:05:35.948634560 +0000
@@ -16,14 +16,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/HUDManager.cs Out/Assembly-CSharp/HUDManager.cs
--- a/Assembly-CSharp/HUDManager.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/HUDManager.cs	2024-05-14 01:05:35.780634555 +0000
@@ -2077,7 +2077,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -2232,7 +2232,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/HauntedMaskItem.cs Out/Assembly-CSharp/HauntedMaskItem.cs
--- a/Assembly-CSharp/HauntedMaskItem.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/HauntedMaskItem.cs	2024-05-14 01:05:35.692634552 +0000
@@ -345,7 +345,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -388,7 +388,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/HoarderBugAI.cs Out/Assembly-CSharp/HoarderBugAI.cs
--- a/Assembly-CSharp/HoarderBugAI.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/HoarderBugAI.cs	2024-05-14 01:05:35.568634548 +0000
@@ -353,7 +353,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -424,7 +424,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/InteractTrigger.cs Out/Assembly-CSharp/InteractTrigger.cs
--- a/Assembly-CSharp/InteractTrigger.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/InteractTrigger.cs	2024-05-14 01:05:35.692634552 +0000
@@ -511,7 +511,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -554,7 +554,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ItemCharger.cs Out/Assembly-CSharp/ItemCharger.cs
--- a/Assembly-CSharp/ItemCharger.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/ItemCharger.cs	2024-05-14 01:05:35.548634547 +0000
@@ -132,7 +132,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -161,7 +161,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ItemDropship.cs Out/Assembly-CSharp/ItemDropship.cs
--- a/Assembly-CSharp/ItemDropship.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/ItemDropship.cs	2024-05-14 01:05:35.692634552 +0000
@@ -128,7 +128,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -171,7 +171,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/JesterAI.cs Out/Assembly-CSharp/JesterAI.cs
--- a/Assembly-CSharp/JesterAI.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/JesterAI.cs	2024-05-14 01:05:35.620634550 +0000
@@ -224,7 +224,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -260,7 +260,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/JetpackItem.cs Out/Assembly-CSharp/JetpackItem.cs
--- a/Assembly-CSharp/JetpackItem.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/JetpackItem.cs	2024-05-14 01:05:35.780634555 +0000
@@ -161,7 +161,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -190,7 +190,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/KeyItem.cs Out/Assembly-CSharp/KeyItem.cs
--- a/Assembly-CSharp/KeyItem.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/KeyItem.cs	2024-05-14 01:05:35.912634559 +0000
@@ -20,14 +20,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/KnifeItem.cs Out/Assembly-CSharp/KnifeItem.cs
--- a/Assembly-CSharp/KnifeItem.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/KnifeItem.cs	2024-05-14 01:05:35.780634555 +0000
@@ -93,7 +93,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -122,7 +122,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/Landmine.cs Out/Assembly-CSharp/Landmine.cs
--- a/Assembly-CSharp/Landmine.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/Landmine.cs	2024-05-14 01:05:35.568634548 +0000
@@ -321,7 +321,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -378,7 +378,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/LassoManAI.cs Out/Assembly-CSharp/LassoManAI.cs
--- a/Assembly-CSharp/LassoManAI.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/LassoManAI.cs	2024-05-14 01:05:35.692634552 +0000
@@ -111,7 +111,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -154,7 +154,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/LockPicker.cs Out/Assembly-CSharp/LockPicker.cs
--- a/Assembly-CSharp/LockPicker.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/LockPicker.cs	2024-05-14 01:05:35.912634559 +0000
@@ -194,7 +194,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -230,7 +230,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/LungProp.cs Out/Assembly-CSharp/LungProp.cs
--- a/Assembly-CSharp/LungProp.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/LungProp.cs	2024-05-14 01:05:35.780634555 +0000
@@ -134,14 +134,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ManualCameraRenderer.cs Out/Assembly-CSharp/ManualCameraRenderer.cs
--- a/Assembly-CSharp/ManualCameraRenderer.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/ManualCameraRenderer.cs	2024-05-14 01:05:35.500634546 +0000
@@ -304,7 +304,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -347,7 +347,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/MapDevice.cs Out/Assembly-CSharp/MapDevice.cs
--- a/Assembly-CSharp/MapDevice.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/MapDevice.cs	2024-05-14 01:05:35.692634552 +0000
@@ -123,14 +123,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/MaskedPlayerEnemy.cs Out/Assembly-CSharp/MaskedPlayerEnemy.cs
--- a/Assembly-CSharp/MaskedPlayerEnemy.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/MaskedPlayerEnemy.cs	2024-05-14 01:05:35.500634546 +0000
@@ -838,7 +838,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -1007,7 +1007,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/MouthDogAI.cs Out/Assembly-CSharp/MouthDogAI.cs
--- a/Assembly-CSharp/MouthDogAI.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/MouthDogAI.cs	2024-05-14 01:05:35.948634560 +0000
@@ -527,7 +527,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -591,7 +591,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/NetworkRigidbodyModifiable.cs Out/Assembly-CSharp/NetworkRigidbodyModifiable.cs
--- a/Assembly-CSharp/NetworkRigidbodyModifiable.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/NetworkRigidbodyModifiable.cs	2024-05-14 01:05:35.780634555 +0000
@@ -80,14 +80,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/NoisemakerProp.cs Out/Assembly-CSharp/NoisemakerProp.cs
--- a/Assembly-CSharp/NoisemakerProp.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/NoisemakerProp.cs	2024-05-14 01:05:35.780634555 +0000
@@ -53,14 +53,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/NutcrackerEnemyAI.cs Out/Assembly-CSharp/NutcrackerEnemyAI.cs
--- a/Assembly-CSharp/NutcrackerEnemyAI.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/NutcrackerEnemyAI.cs	2024-05-14 01:05:35.780634555 +0000
@@ -782,7 +782,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -923,7 +923,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/PatcherTool.cs Out/Assembly-CSharp/PatcherTool.cs
--- a/Assembly-CSharp/PatcherTool.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/PatcherTool.cs	2024-05-14 01:05:35.620634550 +0000
@@ -654,7 +654,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -697,7 +697,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/PhysicsProp.cs Out/Assembly-CSharp/PhysicsProp.cs
--- a/Assembly-CSharp/PhysicsProp.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/PhysicsProp.cs	2024-05-14 01:05:35.780634555 +0000
@@ -20,14 +20,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/PlaceableObjectsSurface.cs Out/Assembly-CSharp/PlaceableObjectsSurface.cs
--- a/Assembly-CSharp/PlaceableObjectsSurface.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/PlaceableObjectsSurface.cs	2024-05-14 01:05:35.568634548 +0000
@@ -46,14 +46,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/PufferAI.cs Out/Assembly-CSharp/PufferAI.cs
--- a/Assembly-CSharp/PufferAI.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/PufferAI.cs	2024-05-14 01:05:35.688634552 +0000
@@ -348,7 +348,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -405,7 +405,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/RadMechAI.cs Out/Assembly-CSharp/RadMechAI.cs
--- a/Assembly-CSharp/RadMechAI.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/RadMechAI.cs	2024-05-14 01:05:35.948634560 +0000
@@ -939,7 +939,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -1087,7 +1087,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/RadarBoosterItem.cs Out/Assembly-CSharp/RadarBoosterItem.cs
--- a/Assembly-CSharp/RadarBoosterItem.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/RadarBoosterItem.cs	2024-05-14 01:05:35.692634552 +0000
@@ -196,7 +196,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -239,7 +239,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/RagdollGrabbableObject.cs Out/Assembly-CSharp/RagdollGrabbableObject.cs
--- a/Assembly-CSharp/RagdollGrabbableObject.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/RagdollGrabbableObject.cs	2024-05-14 01:05:35.780634555 +0000
@@ -91,14 +91,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/RandomPeriodicAudioPlayer.cs Out/Assembly-CSharp/RandomPeriodicAudioPlayer.cs
--- a/Assembly-CSharp/RandomPeriodicAudioPlayer.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/RandomPeriodicAudioPlayer.cs	2024-05-14 01:05:35.780634555 +0000
@@ -55,7 +55,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -77,7 +77,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/RedLocustBees.cs Out/Assembly-CSharp/RedLocustBees.cs
--- a/Assembly-CSharp/RedLocustBees.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/RedLocustBees.cs	2024-05-14 01:05:35.692634552 +0000
@@ -453,7 +453,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -517,7 +517,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/RedPillAnomaly.cs Out/Assembly-CSharp/RedPillAnomaly.cs
--- a/Assembly-CSharp/RedPillAnomaly.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/RedPillAnomaly.cs	2024-05-14 01:05:35.692634552 +0000
@@ -14,14 +14,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/RemoteProp.cs Out/Assembly-CSharp/RemoteProp.cs
--- a/Assembly-CSharp/RemoteProp.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/RemoteProp.cs	2024-05-14 01:05:35.548634547 +0000
@@ -23,14 +23,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/RoundManager.cs Out/Assembly-CSharp/RoundManager.cs
--- a/Assembly-CSharp/RoundManager.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/RoundManager.cs	2024-05-14 01:05:35.780634555 +0000
@@ -1322,7 +1322,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -1442,7 +1442,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/SandSpiderAI.cs Out/Assembly-CSharp/SandSpiderAI.cs
--- a/Assembly-CSharp/SandSpiderAI.cs	2024-05-14 01:01:33.188627075 +0000
+++ Out/Assembly-CSharp/SandSpiderAI.cs	2024-05-14 01:05:35.568634548 +0000
@@ -611,7 +611,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -752,7 +752,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/SandWormAI.cs Out/Assembly-CSharp/SandWormAI.cs
--- a/Assembly-CSharp/SandWormAI.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/SandWormAI.cs	2024-05-14 01:05:35.912634559 +0000
@@ -217,7 +217,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -246,7 +246,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ShipAlarmCord.cs Out/Assembly-CSharp/ShipAlarmCord.cs
--- a/Assembly-CSharp/ShipAlarmCord.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/ShipAlarmCord.cs	2024-05-14 01:05:35.548634547 +0000
@@ -103,7 +103,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -146,7 +146,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ShipBuildModeManager.cs Out/Assembly-CSharp/ShipBuildModeManager.cs
--- a/Assembly-CSharp/ShipBuildModeManager.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/ShipBuildModeManager.cs	2024-05-14 01:05:35.620634550 +0000
@@ -217,7 +217,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -260,7 +260,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ShipLights.cs Out/Assembly-CSharp/ShipLights.cs
--- a/Assembly-CSharp/ShipLights.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/ShipLights.cs	2024-05-14 01:05:35.692634552 +0000
@@ -58,7 +58,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -87,7 +87,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ShipTeleporter.cs Out/Assembly-CSharp/ShipTeleporter.cs
--- a/Assembly-CSharp/ShipTeleporter.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/ShipTeleporter.cs	2024-05-14 01:05:35.780634555 +0000
@@ -419,7 +419,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -476,7 +476,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ShotgunItem.cs Out/Assembly-CSharp/ShotgunItem.cs
--- a/Assembly-CSharp/ShotgunItem.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/ShotgunItem.cs	2024-05-14 01:05:35.692634552 +0000
@@ -376,7 +376,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -419,7 +419,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/Shovel.cs Out/Assembly-CSharp/Shovel.cs
--- a/Assembly-CSharp/Shovel.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/Shovel.cs	2024-05-14 01:05:35.948634560 +0000
@@ -183,7 +183,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -226,7 +226,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/SoundManager.cs Out/Assembly-CSharp/SoundManager.cs
--- a/Assembly-CSharp/SoundManager.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/SoundManager.cs	2024-05-14 01:05:35.780634555 +0000
@@ -453,7 +453,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -496,7 +496,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/SpikeRoofTrap.cs Out/Assembly-CSharp/SpikeRoofTrap.cs
--- a/Assembly-CSharp/SpikeRoofTrap.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/SpikeRoofTrap.cs	2024-05-14 01:05:35.620634550 +0000
@@ -210,7 +210,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -253,7 +253,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/SprayPaintItem.cs Out/Assembly-CSharp/SprayPaintItem.cs
--- a/Assembly-CSharp/SprayPaintItem.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/SprayPaintItem.cs	2024-05-14 01:05:35.780634555 +0000
@@ -214,7 +214,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -243,7 +243,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/SpringManAI.cs Out/Assembly-CSharp/SpringManAI.cs
--- a/Assembly-CSharp/SpringManAI.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/SpringManAI.cs	2024-05-14 01:05:35.912634559 +0000
@@ -101,7 +101,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -144,7 +144,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/StartMatchLever.cs Out/Assembly-CSharp/StartMatchLever.cs
--- a/Assembly-CSharp/StartMatchLever.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/StartMatchLever.cs	2024-05-14 01:05:35.948634560 +0000
@@ -114,7 +114,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -150,7 +150,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/StartOfRound.cs Out/Assembly-CSharp/StartOfRound.cs
--- a/Assembly-CSharp/StartOfRound.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/StartOfRound.cs	2024-05-14 01:05:35.948634560 +0000
@@ -2481,7 +2481,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -2783,7 +2783,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/SteamValveHazard.cs Out/Assembly-CSharp/SteamValveHazard.cs
--- a/Assembly-CSharp/SteamValveHazard.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/SteamValveHazard.cs	2024-05-14 01:05:35.620634550 +0000
@@ -110,7 +110,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -139,7 +139,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/StoryLog.cs Out/Assembly-CSharp/StoryLog.cs
--- a/Assembly-CSharp/StoryLog.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/StoryLog.cs	2024-05-14 01:05:51.484635039 +0000
@@ -46,14 +46,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/StunGrenadeItem.cs Out/Assembly-CSharp/StunGrenadeItem.cs
--- a/Assembly-CSharp/StunGrenadeItem.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/StunGrenadeItem.cs	2024-05-14 01:05:35.948634560 +0000
@@ -239,7 +239,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -268,7 +268,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/SyncWithServerOnSpawn.cs Out/Assembly-CSharp/SyncWithServerOnSpawn.cs
--- a/Assembly-CSharp/SyncWithServerOnSpawn.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/SyncWithServerOnSpawn.cs	2024-05-14 01:05:35.780634555 +0000
@@ -33,14 +33,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/TVScript.cs Out/Assembly-CSharp/TVScript.cs
--- a/Assembly-CSharp/TVScript.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/TVScript.cs	2024-05-14 01:05:35.948634560 +0000
@@ -156,7 +156,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -220,7 +220,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/Terminal.cs Out/Assembly-CSharp/Terminal.cs
--- a/Assembly-CSharp/Terminal.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/Terminal.cs	2024-05-14 01:05:35.620634550 +0000
@@ -978,7 +978,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -1056,7 +1056,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/TerminalAccessibleObject.cs Out/Assembly-CSharp/TerminalAccessibleObject.cs
--- a/Assembly-CSharp/TerminalAccessibleObject.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/TerminalAccessibleObject.cs	2024-05-14 01:05:35.488634546 +0000
@@ -233,7 +233,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -262,7 +262,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/TestEnemy.cs Out/Assembly-CSharp/TestEnemy.cs
--- a/Assembly-CSharp/TestEnemy.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/TestEnemy.cs	2024-05-14 01:05:35.948634560 +0000
@@ -49,14 +49,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/TetraChemicalItem.cs Out/Assembly-CSharp/TetraChemicalItem.cs
--- a/Assembly-CSharp/TetraChemicalItem.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/TetraChemicalItem.cs	2024-05-14 01:05:35.780634555 +0000
@@ -159,7 +159,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -188,7 +188,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/TimeOfDay.cs Out/Assembly-CSharp/TimeOfDay.cs
--- a/Assembly-CSharp/TimeOfDay.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/TimeOfDay.cs	2024-05-14 01:05:35.780634555 +0000
@@ -533,7 +533,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -597,7 +597,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/Turret.cs Out/Assembly-CSharp/Turret.cs
--- a/Assembly-CSharp/Turret.cs	2024-05-14 01:01:33.192627075 +0000
+++ Out/Assembly-CSharp/Turret.cs	2024-05-14 01:05:35.488634546 +0000
@@ -331,7 +331,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -402,7 +402,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/UnlockableSuit.cs Out/Assembly-CSharp/UnlockableSuit.cs
--- a/Assembly-CSharp/UnlockableSuit.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/UnlockableSuit.cs	2024-05-14 01:05:35.500634546 +0000
@@ -70,7 +70,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -99,7 +99,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/WalkieTalkie.cs Out/Assembly-CSharp/WalkieTalkie.cs
--- a/Assembly-CSharp/WalkieTalkie.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/WalkieTalkie.cs	2024-05-14 01:05:35.620634550 +0000
@@ -347,7 +347,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -390,7 +390,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/WhoopieCushionItem.cs Out/Assembly-CSharp/WhoopieCushionItem.cs
--- a/Assembly-CSharp/WhoopieCushionItem.cs	2024-05-14 01:01:33.196627075 +0000
+++ Out/Assembly-CSharp/WhoopieCushionItem.cs	2024-05-14 01:05:35.692634552 +0000
@@ -40,14 +40,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur a/ClientNetworkTransform/ClientNetworkTransform.csproj Out/ClientNetworkTransform/ClientNetworkTransform.csproj
--- a/ClientNetworkTransform/ClientNetworkTransform.csproj	2024-05-14 01:01:33.212627076 +0000
+++ Out/ClientNetworkTransform/ClientNetworkTransform.csproj	2024-05-14 01:09:55.976642577 +0000
@@ -14,6 +14,9 @@
     <Reference Include="Unity.Netcode.Components">
       <HintPath>../../Temp/Assemblies/Unity.Netcode.Components.dll</HintPath>
     </Reference>
+    <Reference Include="Unity.Netcode.Runtime">
+      <HintPath>../../Temp/Assemblies/Unity.Netcode.Runtime.dll</HintPath>
+    </Reference>
     <Reference Include="UnityEngine.CoreModule">
       <HintPath>../../Temp/Assemblies/UnityEngine.CoreModule.dll</HintPath>
     </Reference>
