diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/AnimatedItem.cs Out/Assembly-CSharp/AnimatedItem.cs
--- Temp/PatchSource/Assembly-CSharp/AnimatedItem.cs	2024-09-10 05:42:03.306153791 +0000
+++ Out/Assembly-CSharp/AnimatedItem.cs	2024-09-10 05:40:43.530150984 +0000
@@ -95,14 +95,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/AnimatedObjectTrigger.cs Out/Assembly-CSharp/AnimatedObjectTrigger.cs
--- Temp/PatchSource/Assembly-CSharp/AnimatedObjectTrigger.cs	2024-09-10 05:42:03.294153791 +0000
+++ Out/Assembly-CSharp/AnimatedObjectTrigger.cs	2024-09-10 05:40:43.534150984 +0000
@@ -244,7 +244,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -287,7 +287,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/Anomaly.cs Out/Assembly-CSharp/Anomaly.cs
--- Temp/PatchSource/Assembly-CSharp/Anomaly.cs	2024-09-10 05:42:03.326153792 +0000
+++ Out/Assembly-CSharp/Anomaly.cs	2024-09-10 05:40:43.326150977 +0000
@@ -71,7 +71,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -93,7 +93,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/AutoParentToShip.cs Out/Assembly-CSharp/AutoParentToShip.cs
--- Temp/PatchSource/Assembly-CSharp/AutoParentToShip.cs	2024-09-10 05:42:03.342153793 +0000
+++ Out/Assembly-CSharp/AutoParentToShip.cs	2024-09-10 05:40:43.326150977 +0000
@@ -135,14 +135,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/BaboonBirdAI.cs Out/Assembly-CSharp/BaboonBirdAI.cs
--- Temp/PatchSource/Assembly-CSharp/BaboonBirdAI.cs	2024-09-10 05:42:03.326153792 +0000
+++ Out/Assembly-CSharp/BaboonBirdAI.cs	2024-09-10 05:40:43.326150977 +0000
@@ -780,7 +780,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -1005,7 +1005,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/BinocularsItem.cs Out/Assembly-CSharp/BinocularsItem.cs
--- Temp/PatchSource/Assembly-CSharp/BinocularsItem.cs	2024-09-10 05:42:03.278153790 +0000
+++ Out/Assembly-CSharp/BinocularsItem.cs	2024-09-10 05:40:43.538150984 +0000
@@ -14,14 +14,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/BlobAI.cs Out/Assembly-CSharp/BlobAI.cs
--- Temp/PatchSource/Assembly-CSharp/BlobAI.cs	2024-09-10 05:42:03.362153793 +0000
+++ Out/Assembly-CSharp/BlobAI.cs	2024-09-10 05:40:43.238150974 +0000
@@ -274,7 +274,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -303,7 +303,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/BoomboxItem.cs Out/Assembly-CSharp/BoomboxItem.cs
--- Temp/PatchSource/Assembly-CSharp/BoomboxItem.cs	2024-09-10 05:42:03.286153791 +0000
+++ Out/Assembly-CSharp/BoomboxItem.cs	2024-09-10 05:40:43.538150984 +0000
@@ -151,14 +151,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/BreakerBox.cs Out/Assembly-CSharp/BreakerBox.cs
--- Temp/PatchSource/Assembly-CSharp/BreakerBox.cs	2024-09-10 05:42:03.314153792 +0000
+++ Out/Assembly-CSharp/BreakerBox.cs	2024-09-10 05:40:43.330150977 +0000
@@ -99,14 +99,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/BridgeTrigger.cs Out/Assembly-CSharp/BridgeTrigger.cs
--- Temp/PatchSource/Assembly-CSharp/BridgeTrigger.cs	2024-09-10 05:42:03.314153792 +0000
+++ Out/Assembly-CSharp/BridgeTrigger.cs	2024-09-10 05:40:43.330150977 +0000
@@ -122,7 +122,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -151,7 +151,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/BridgeTriggerType2.cs Out/Assembly-CSharp/BridgeTriggerType2.cs
--- Temp/PatchSource/Assembly-CSharp/BridgeTriggerType2.cs	2024-09-10 05:42:03.354153793 +0000
+++ Out/Assembly-CSharp/BridgeTriggerType2.cs	2024-09-10 05:40:43.242150974 +0000
@@ -38,7 +38,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -60,7 +60,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/BushWolfEnemy.cs Out/Assembly-CSharp/BushWolfEnemy.cs
--- Temp/PatchSource/Assembly-CSharp/BushWolfEnemy.cs	2024-09-10 05:42:03.346153793 +0000
+++ Out/Assembly-CSharp/BushWolfEnemy.cs	2024-09-10 05:40:43.326150977 +0000
@@ -742,7 +742,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -911,7 +911,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/ButlerBeesEnemyAI.cs Out/Assembly-CSharp/ButlerBeesEnemyAI.cs
--- Temp/PatchSource/Assembly-CSharp/ButlerBeesEnemyAI.cs	2024-09-10 05:42:03.266153790 +0000
+++ Out/Assembly-CSharp/ButlerBeesEnemyAI.cs	2024-09-10 05:40:43.538150984 +0000
@@ -46,14 +46,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/ButlerEnemyAI.cs Out/Assembly-CSharp/ButlerEnemyAI.cs
--- Temp/PatchSource/Assembly-CSharp/ButlerEnemyAI.cs	2024-09-10 05:42:03.274153790 +0000
+++ Out/Assembly-CSharp/ButlerEnemyAI.cs	2024-09-10 05:40:43.538150984 +0000
@@ -660,7 +660,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -843,7 +843,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/CaveDwellerAI.cs Out/Assembly-CSharp/CaveDwellerAI.cs
--- Temp/PatchSource/Assembly-CSharp/CaveDwellerAI.cs	2024-09-10 05:42:03.266153790 +0000
+++ Out/Assembly-CSharp/CaveDwellerAI.cs	2024-09-10 05:40:43.538150984 +0000
@@ -1188,7 +1188,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -1504,7 +1504,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/CaveDwellerPhysicsProp.cs Out/Assembly-CSharp/CaveDwellerPhysicsProp.cs
--- Temp/PatchSource/Assembly-CSharp/CaveDwellerPhysicsProp.cs	2024-09-10 05:42:03.290153791 +0000
+++ Out/Assembly-CSharp/CaveDwellerPhysicsProp.cs	2024-09-10 05:40:43.538150984 +0000
@@ -114,7 +114,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -171,7 +171,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/CentipedeAI.cs Out/Assembly-CSharp/CentipedeAI.cs
--- Temp/PatchSource/Assembly-CSharp/CentipedeAI.cs	2024-09-10 05:42:03.366153793 +0000
+++ Out/Assembly-CSharp/CentipedeAI.cs	2024-09-10 05:40:43.530150984 +0000
@@ -618,7 +618,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -696,7 +696,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/ClaySurgeonAI.cs Out/Assembly-CSharp/ClaySurgeonAI.cs
--- Temp/PatchSource/Assembly-CSharp/ClaySurgeonAI.cs	2024-09-10 05:42:03.302153791 +0000
+++ Out/Assembly-CSharp/ClaySurgeonAI.cs	2024-09-10 05:40:43.534150984 +0000
@@ -260,7 +260,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -310,7 +310,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/ClipboardItem.cs Out/Assembly-CSharp/ClipboardItem.cs
--- Temp/PatchSource/Assembly-CSharp/ClipboardItem.cs	2024-09-10 05:42:03.358153793 +0000
+++ Out/Assembly-CSharp/ClipboardItem.cs	2024-09-10 05:40:43.238150974 +0000
@@ -58,14 +58,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/ClockProp.cs Out/Assembly-CSharp/ClockProp.cs
--- Temp/PatchSource/Assembly-CSharp/ClockProp.cs	2024-09-10 05:42:03.326153792 +0000
+++ Out/Assembly-CSharp/ClockProp.cs	2024-09-10 05:40:43.326150977 +0000
@@ -45,14 +45,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/CrawlerAI.cs Out/Assembly-CSharp/CrawlerAI.cs
--- Temp/PatchSource/Assembly-CSharp/CrawlerAI.cs	2024-09-10 05:42:03.334153792 +0000
+++ Out/Assembly-CSharp/CrawlerAI.cs	2024-09-10 05:40:43.326150977 +0000
@@ -320,7 +320,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -405,7 +405,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/DepositItemsDesk.cs Out/Assembly-CSharp/DepositItemsDesk.cs
--- Temp/PatchSource/Assembly-CSharp/DepositItemsDesk.cs	2024-09-10 05:42:03.302153791 +0000
+++ Out/Assembly-CSharp/DepositItemsDesk.cs	2024-09-10 05:40:43.534150984 +0000
@@ -652,7 +652,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -765,7 +765,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/DigitalRuby/ThunderAndLightning/SingleLineClampAttribute.cs Out/Assembly-CSharp/DigitalRuby/ThunderAndLightning/SingleLineClampAttribute.cs
--- Temp/PatchSource/Assembly-CSharp/DigitalRuby/ThunderAndLightning/SingleLineClampAttribute.cs	2024-09-10 05:41:58.350153617 +0000
+++ Out/Assembly-CSharp/DigitalRuby/ThunderAndLightning/SingleLineClampAttribute.cs	2024-09-10 05:41:21.834152332 +0000
@@ -41,7 +41,7 @@
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
-	public SingleLineClampAttribute(string tooltip, double minValue, double maxValue)
+	public SingleLineClampAttribute(string tooltip, double minValue, double maxValue) : base(tooltip)
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/DocileLocustBeesAI.cs Out/Assembly-CSharp/DocileLocustBeesAI.cs
--- Temp/PatchSource/Assembly-CSharp/DocileLocustBeesAI.cs	2024-09-10 05:42:03.270153790 +0000
+++ Out/Assembly-CSharp/DocileLocustBeesAI.cs	2024-09-10 05:40:43.538150984 +0000
@@ -120,14 +120,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/DoorLock.cs Out/Assembly-CSharp/DoorLock.cs
--- Temp/PatchSource/Assembly-CSharp/DoorLock.cs	2024-09-10 05:42:03.270153790 +0000
+++ Out/Assembly-CSharp/DoorLock.cs	2024-09-10 05:40:43.538150984 +0000
@@ -224,7 +224,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -309,7 +309,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/DoublewingAI.cs Out/Assembly-CSharp/DoublewingAI.cs
--- Temp/PatchSource/Assembly-CSharp/DoublewingAI.cs	2024-09-10 05:42:03.338153793 +0000
+++ Out/Assembly-CSharp/DoublewingAI.cs	2024-09-10 05:40:43.326150977 +0000
@@ -293,7 +293,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -329,7 +329,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/DressGirlAI.cs Out/Assembly-CSharp/DressGirlAI.cs
--- Temp/PatchSource/Assembly-CSharp/DressGirlAI.cs	2024-09-10 05:42:03.358153793 +0000
+++ Out/Assembly-CSharp/DressGirlAI.cs	2024-09-10 05:40:43.238150974 +0000
@@ -362,7 +362,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -412,7 +412,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/EnemyAI.cs Out/Assembly-CSharp/EnemyAI.cs
--- Temp/PatchSource/Assembly-CSharp/EnemyAI.cs	2024-09-10 05:42:03.370153794 +0000
+++ Out/Assembly-CSharp/EnemyAI.cs	2024-09-10 05:40:43.178150972 +0000
@@ -773,7 +773,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -872,7 +872,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/EnemyVent.cs Out/Assembly-CSharp/EnemyVent.cs
--- Temp/PatchSource/Assembly-CSharp/EnemyVent.cs	2024-09-10 05:42:03.358153793 +0000
+++ Out/Assembly-CSharp/EnemyVent.cs	2024-09-10 05:40:43.238150974 +0000
@@ -81,7 +81,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -110,7 +110,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/EntranceTeleport.cs Out/Assembly-CSharp/EntranceTeleport.cs
--- Temp/PatchSource/Assembly-CSharp/EntranceTeleport.cs	2024-09-10 05:42:03.298153791 +0000
+++ Out/Assembly-CSharp/EntranceTeleport.cs	2024-09-10 05:40:43.534150984 +0000
@@ -178,7 +178,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -207,7 +207,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/EventWhenDroppedItem.cs Out/Assembly-CSharp/EventWhenDroppedItem.cs
--- Temp/PatchSource/Assembly-CSharp/EventWhenDroppedItem.cs	2024-09-10 05:42:03.322153792 +0000
+++ Out/Assembly-CSharp/EventWhenDroppedItem.cs	2024-09-10 05:40:43.326150977 +0000
@@ -48,14 +48,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/ExtensionLadderItem.cs Out/Assembly-CSharp/ExtensionLadderItem.cs
--- Temp/PatchSource/Assembly-CSharp/ExtensionLadderItem.cs	2024-09-10 05:42:03.266153790 +0000
+++ Out/Assembly-CSharp/ExtensionLadderItem.cs	2024-09-10 05:40:43.538150984 +0000
@@ -215,14 +215,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/FlashlightItem.cs Out/Assembly-CSharp/FlashlightItem.cs
--- Temp/PatchSource/Assembly-CSharp/FlashlightItem.cs	2024-09-10 05:42:03.290153791 +0000
+++ Out/Assembly-CSharp/FlashlightItem.cs	2024-09-10 05:40:43.538150984 +0000
@@ -114,7 +114,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -143,7 +143,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/FlowerSnakeEnemy.cs Out/Assembly-CSharp/FlowerSnakeEnemy.cs
--- Temp/PatchSource/Assembly-CSharp/FlowerSnakeEnemy.cs	2024-09-10 05:42:03.358153793 +0000
+++ Out/Assembly-CSharp/FlowerSnakeEnemy.cs	2024-09-10 05:40:43.238150974 +0000
@@ -424,7 +424,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -523,7 +523,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/FlowermanAI.cs Out/Assembly-CSharp/FlowermanAI.cs
--- Temp/PatchSource/Assembly-CSharp/FlowermanAI.cs	2024-09-10 05:42:03.302153791 +0000
+++ Out/Assembly-CSharp/FlowermanAI.cs	2024-09-10 05:40:43.534150984 +0000
@@ -372,7 +372,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -450,7 +450,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/ForestGiantAI.cs Out/Assembly-CSharp/ForestGiantAI.cs
--- Temp/PatchSource/Assembly-CSharp/ForestGiantAI.cs	2024-09-10 05:42:03.358153793 +0000
+++ Out/Assembly-CSharp/ForestGiantAI.cs	2024-09-10 05:40:43.238150974 +0000
@@ -460,7 +460,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -524,7 +524,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/GameNetcodeStuff/PlayerControllerB.cs Out/Assembly-CSharp/GameNetcodeStuff/PlayerControllerB.cs
--- Temp/PatchSource/Assembly-CSharp/GameNetcodeStuff/PlayerControllerB.cs	2024-09-10 05:42:00.186153682 +0000
+++ Out/Assembly-CSharp/GameNetcodeStuff/PlayerControllerB.cs	2024-09-10 05:40:43.938150999 +0000
@@ -12,6 +12,7 @@
 using UnityEngine.Animations.Rigging;
 using UnityEngine.InputSystem;
 using UnityEngine.UI;
+using DefaultNamespace;
 
 namespace GameNetcodeStuff;
 
@@ -2648,7 +2649,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -3097,7 +3098,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/GiftBoxItem.cs Out/Assembly-CSharp/GiftBoxItem.cs
--- Temp/PatchSource/Assembly-CSharp/GiftBoxItem.cs	2024-09-10 05:42:03.290153791 +0000
+++ Out/Assembly-CSharp/GiftBoxItem.cs	2024-09-10 05:40:43.538150984 +0000
@@ -254,7 +254,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -290,7 +290,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/GlobalEffects.cs Out/Assembly-CSharp/GlobalEffects.cs
--- Temp/PatchSource/Assembly-CSharp/GlobalEffects.cs	2024-09-10 05:42:03.318153792 +0000
+++ Out/Assembly-CSharp/GlobalEffects.cs	2024-09-10 05:40:43.330150977 +0000
@@ -129,7 +129,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -186,7 +186,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/GrabbableObject.cs Out/Assembly-CSharp/GrabbableObject.cs
--- Temp/PatchSource/Assembly-CSharp/GrabbableObject.cs	2024-09-10 05:42:03.302153791 +0000
+++ Out/Assembly-CSharp/GrabbableObject.cs	2024-09-10 05:40:43.530150984 +0000
@@ -452,7 +452,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -586,7 +586,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/GunAmmo.cs Out/Assembly-CSharp/GunAmmo.cs
--- Temp/PatchSource/Assembly-CSharp/GunAmmo.cs	2024-09-10 05:42:03.330153792 +0000
+++ Out/Assembly-CSharp/GunAmmo.cs	2024-09-10 05:40:43.326150977 +0000
@@ -16,14 +16,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/HUDManager.cs Out/Assembly-CSharp/HUDManager.cs
--- Temp/PatchSource/Assembly-CSharp/HUDManager.cs	2024-09-10 05:42:03.278153790 +0000
+++ Out/Assembly-CSharp/HUDManager.cs	2024-09-10 05:40:43.538150984 +0000
@@ -2077,7 +2077,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -2232,7 +2232,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/HauntedMaskItem.cs Out/Assembly-CSharp/HauntedMaskItem.cs
--- Temp/PatchSource/Assembly-CSharp/HauntedMaskItem.cs	2024-09-10 05:42:03.270153790 +0000
+++ Out/Assembly-CSharp/HauntedMaskItem.cs	2024-09-10 05:40:43.538150984 +0000
@@ -345,7 +345,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -388,7 +388,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/HoarderBugAI.cs Out/Assembly-CSharp/HoarderBugAI.cs
--- Temp/PatchSource/Assembly-CSharp/HoarderBugAI.cs	2024-09-10 05:42:03.362153793 +0000
+++ Out/Assembly-CSharp/HoarderBugAI.cs	2024-09-10 05:40:43.238150974 +0000
@@ -353,7 +353,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -424,7 +424,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/InteractTrigger.cs Out/Assembly-CSharp/InteractTrigger.cs
--- Temp/PatchSource/Assembly-CSharp/InteractTrigger.cs	2024-09-10 05:42:03.286153791 +0000
+++ Out/Assembly-CSharp/InteractTrigger.cs	2024-09-10 05:40:43.170150972 +0000
@@ -534,7 +534,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -577,7 +577,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/ItemCharger.cs Out/Assembly-CSharp/ItemCharger.cs
--- Temp/PatchSource/Assembly-CSharp/ItemCharger.cs	2024-09-10 05:42:03.362153793 +0000
+++ Out/Assembly-CSharp/ItemCharger.cs	2024-09-10 05:40:43.178150972 +0000
@@ -132,7 +132,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -161,7 +161,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/ItemDropship.cs Out/Assembly-CSharp/ItemDropship.cs
--- Temp/PatchSource/Assembly-CSharp/ItemDropship.cs	2024-09-10 05:42:03.330153792 +0000
+++ Out/Assembly-CSharp/ItemDropship.cs	2024-09-10 05:40:43.326150977 +0000
@@ -193,7 +193,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -271,7 +271,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/JesterAI.cs Out/Assembly-CSharp/JesterAI.cs
--- Temp/PatchSource/Assembly-CSharp/JesterAI.cs	2024-09-10 05:42:03.350153793 +0000
+++ Out/Assembly-CSharp/JesterAI.cs	2024-09-10 05:40:43.242150974 +0000
@@ -224,7 +224,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -260,7 +260,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/JetpackItem.cs Out/Assembly-CSharp/JetpackItem.cs
--- Temp/PatchSource/Assembly-CSharp/JetpackItem.cs	2024-09-10 05:42:03.282153791 +0000
+++ Out/Assembly-CSharp/JetpackItem.cs	2024-09-10 05:40:43.538150984 +0000
@@ -161,7 +161,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -190,7 +190,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/KeyItem.cs Out/Assembly-CSharp/KeyItem.cs
--- Temp/PatchSource/Assembly-CSharp/KeyItem.cs	2024-09-10 05:42:03.330153792 +0000
+++ Out/Assembly-CSharp/KeyItem.cs	2024-09-10 05:40:43.326150977 +0000
@@ -20,14 +20,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/KnifeItem.cs Out/Assembly-CSharp/KnifeItem.cs
--- Temp/PatchSource/Assembly-CSharp/KnifeItem.cs	2024-09-10 05:42:03.302153791 +0000
+++ Out/Assembly-CSharp/KnifeItem.cs	2024-09-10 05:40:43.534150984 +0000
@@ -93,7 +93,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -122,7 +122,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/Landmine.cs Out/Assembly-CSharp/Landmine.cs
--- Temp/PatchSource/Assembly-CSharp/Landmine.cs	2024-09-10 05:42:03.286153791 +0000
+++ Out/Assembly-CSharp/Landmine.cs	2024-09-10 05:40:43.538150984 +0000
@@ -321,7 +321,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -378,7 +378,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/LassoManAI.cs Out/Assembly-CSharp/LassoManAI.cs
--- Temp/PatchSource/Assembly-CSharp/LassoManAI.cs	2024-09-10 05:42:03.310153792 +0000
+++ Out/Assembly-CSharp/LassoManAI.cs	2024-09-10 05:40:43.530150984 +0000
@@ -111,7 +111,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -154,7 +154,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/LockPicker.cs Out/Assembly-CSharp/LockPicker.cs
--- Temp/PatchSource/Assembly-CSharp/LockPicker.cs	2024-09-10 05:42:03.290153791 +0000
+++ Out/Assembly-CSharp/LockPicker.cs	2024-09-10 05:40:43.538150984 +0000
@@ -194,7 +194,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -230,7 +230,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/LungProp.cs Out/Assembly-CSharp/LungProp.cs
--- Temp/PatchSource/Assembly-CSharp/LungProp.cs	2024-09-10 05:42:03.362153793 +0000
+++ Out/Assembly-CSharp/LungProp.cs	2024-09-10 05:40:43.178150972 +0000
@@ -134,14 +134,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/ManualCameraRenderer.cs Out/Assembly-CSharp/ManualCameraRenderer.cs
--- Temp/PatchSource/Assembly-CSharp/ManualCameraRenderer.cs	2024-09-10 05:42:03.318153792 +0000
+++ Out/Assembly-CSharp/ManualCameraRenderer.cs	2024-09-10 05:40:43.330150977 +0000
@@ -304,7 +304,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -347,7 +347,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/MapDevice.cs Out/Assembly-CSharp/MapDevice.cs
--- Temp/PatchSource/Assembly-CSharp/MapDevice.cs	2024-09-10 05:42:03.338153793 +0000
+++ Out/Assembly-CSharp/MapDevice.cs	2024-09-10 05:40:43.326150977 +0000
@@ -123,14 +123,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/MaskedPlayerEnemy.cs Out/Assembly-CSharp/MaskedPlayerEnemy.cs
--- Temp/PatchSource/Assembly-CSharp/MaskedPlayerEnemy.cs	2024-09-10 05:42:03.338153793 +0000
+++ Out/Assembly-CSharp/MaskedPlayerEnemy.cs	2024-09-10 05:40:43.330150977 +0000
@@ -858,7 +858,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -1027,7 +1027,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/MineshaftElevatorController.cs Out/Assembly-CSharp/MineshaftElevatorController.cs
--- Temp/PatchSource/Assembly-CSharp/MineshaftElevatorController.cs	2024-09-10 05:42:03.318153792 +0000
+++ Out/Assembly-CSharp/MineshaftElevatorController.cs	2024-09-10 05:40:43.330150977 +0000
@@ -249,7 +249,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -334,7 +334,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/MoldSpreadManager.cs Out/Assembly-CSharp/MoldSpreadManager.cs
--- Temp/PatchSource/Assembly-CSharp/MoldSpreadManager.cs	2024-09-10 05:42:03.338153793 +0000
+++ Out/Assembly-CSharp/MoldSpreadManager.cs	2024-09-10 05:40:43.326150977 +0000
@@ -102,14 +102,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/MouthDogAI.cs Out/Assembly-CSharp/MouthDogAI.cs
--- Temp/PatchSource/Assembly-CSharp/MouthDogAI.cs	2024-09-10 05:42:03.358153793 +0000
+++ Out/Assembly-CSharp/MouthDogAI.cs	2024-09-10 05:40:43.238150974 +0000
@@ -527,7 +527,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -591,7 +591,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/NetworkRigidbodyModifiable.cs Out/Assembly-CSharp/NetworkRigidbodyModifiable.cs
--- Temp/PatchSource/Assembly-CSharp/NetworkRigidbodyModifiable.cs	2024-09-10 05:42:03.278153790 +0000
+++ Out/Assembly-CSharp/NetworkRigidbodyModifiable.cs	2024-09-10 05:40:43.170150972 +0000
@@ -80,14 +80,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/NoisemakerProp.cs Out/Assembly-CSharp/NoisemakerProp.cs
--- Temp/PatchSource/Assembly-CSharp/NoisemakerProp.cs	2024-09-10 05:42:03.270153790 +0000
+++ Out/Assembly-CSharp/NoisemakerProp.cs	2024-09-10 05:40:43.538150984 +0000
@@ -53,14 +53,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/NutcrackerEnemyAI.cs Out/Assembly-CSharp/NutcrackerEnemyAI.cs
--- Temp/PatchSource/Assembly-CSharp/NutcrackerEnemyAI.cs	2024-09-10 05:42:03.346153793 +0000
+++ Out/Assembly-CSharp/NutcrackerEnemyAI.cs	2024-09-10 05:40:43.326150977 +0000
@@ -782,7 +782,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -923,7 +923,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/PatcherTool.cs Out/Assembly-CSharp/PatcherTool.cs
--- Temp/PatchSource/Assembly-CSharp/PatcherTool.cs	2024-09-10 05:42:03.354153793 +0000
+++ Out/Assembly-CSharp/PatcherTool.cs	2024-09-10 05:40:43.242150974 +0000
@@ -654,7 +654,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -697,7 +697,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/PhysicsProp.cs Out/Assembly-CSharp/PhysicsProp.cs
--- Temp/PatchSource/Assembly-CSharp/PhysicsProp.cs	2024-09-10 05:42:03.330153792 +0000
+++ Out/Assembly-CSharp/PhysicsProp.cs	2024-09-10 05:40:43.326150977 +0000
@@ -20,14 +20,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/PlaceableObjectsSurface.cs Out/Assembly-CSharp/PlaceableObjectsSurface.cs
--- Temp/PatchSource/Assembly-CSharp/PlaceableObjectsSurface.cs	2024-09-10 05:42:03.338153793 +0000
+++ Out/Assembly-CSharp/PlaceableObjectsSurface.cs	2024-09-10 05:40:43.326150977 +0000
@@ -46,14 +46,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/PufferAI.cs Out/Assembly-CSharp/PufferAI.cs
--- Temp/PatchSource/Assembly-CSharp/PufferAI.cs	2024-09-10 05:42:03.294153791 +0000
+++ Out/Assembly-CSharp/PufferAI.cs	2024-09-10 05:40:43.538150984 +0000
@@ -354,7 +354,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -411,7 +411,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/RadMechAI.cs Out/Assembly-CSharp/RadMechAI.cs
--- Temp/PatchSource/Assembly-CSharp/RadMechAI.cs	2024-09-10 05:42:03.294153791 +0000
+++ Out/Assembly-CSharp/RadMechAI.cs	2024-09-10 05:40:43.534150984 +0000
@@ -939,7 +939,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -1087,7 +1087,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/RadarBoosterItem.cs Out/Assembly-CSharp/RadarBoosterItem.cs
--- Temp/PatchSource/Assembly-CSharp/RadarBoosterItem.cs	2024-09-10 05:42:03.322153792 +0000
+++ Out/Assembly-CSharp/RadarBoosterItem.cs	2024-09-10 05:40:43.326150977 +0000
@@ -196,7 +196,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -239,7 +239,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/RagdollGrabbableObject.cs Out/Assembly-CSharp/RagdollGrabbableObject.cs
--- Temp/PatchSource/Assembly-CSharp/RagdollGrabbableObject.cs	2024-09-10 05:42:03.262153790 +0000
+++ Out/Assembly-CSharp/RagdollGrabbableObject.cs	2024-09-10 05:40:43.538150984 +0000
@@ -91,14 +91,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/RandomPeriodicAudioPlayer.cs Out/Assembly-CSharp/RandomPeriodicAudioPlayer.cs
--- Temp/PatchSource/Assembly-CSharp/RandomPeriodicAudioPlayer.cs	2024-09-10 05:42:03.346153793 +0000
+++ Out/Assembly-CSharp/RandomPeriodicAudioPlayer.cs	2024-09-10 05:40:43.326150977 +0000
@@ -55,7 +55,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -77,7 +77,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/RedLocustBees.cs Out/Assembly-CSharp/RedLocustBees.cs
--- Temp/PatchSource/Assembly-CSharp/RedLocustBees.cs	2024-09-10 05:42:03.298153791 +0000
+++ Out/Assembly-CSharp/RedLocustBees.cs	2024-09-10 05:40:43.534150984 +0000
@@ -453,7 +453,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -517,7 +517,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/RedPillAnomaly.cs Out/Assembly-CSharp/RedPillAnomaly.cs
--- Temp/PatchSource/Assembly-CSharp/RedPillAnomaly.cs	2024-09-10 05:42:03.326153792 +0000
+++ Out/Assembly-CSharp/RedPillAnomaly.cs	2024-09-10 05:40:43.326150977 +0000
@@ -14,14 +14,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/RemoteProp.cs Out/Assembly-CSharp/RemoteProp.cs
--- Temp/PatchSource/Assembly-CSharp/RemoteProp.cs	2024-09-10 05:42:03.330153792 +0000
+++ Out/Assembly-CSharp/RemoteProp.cs	2024-09-10 05:40:43.326150977 +0000
@@ -23,14 +23,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/RoundManager.cs Out/Assembly-CSharp/RoundManager.cs
--- Temp/PatchSource/Assembly-CSharp/RoundManager.cs	2024-09-10 05:42:03.298153791 +0000
+++ Out/Assembly-CSharp/RoundManager.cs	2024-09-10 05:40:43.534150984 +0000
@@ -1397,7 +1397,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -1531,7 +1531,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/SandSpiderAI.cs Out/Assembly-CSharp/SandSpiderAI.cs
--- Temp/PatchSource/Assembly-CSharp/SandSpiderAI.cs	2024-09-10 05:42:03.290153791 +0000
+++ Out/Assembly-CSharp/SandSpiderAI.cs	2024-09-10 05:40:43.538150984 +0000
@@ -611,7 +611,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -752,7 +752,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/SandWormAI.cs Out/Assembly-CSharp/SandWormAI.cs
--- Temp/PatchSource/Assembly-CSharp/SandWormAI.cs	2024-09-10 05:42:03.366153793 +0000
+++ Out/Assembly-CSharp/SandWormAI.cs	2024-09-10 05:40:43.178150972 +0000
@@ -217,7 +217,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -246,7 +246,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/ShipAlarmCord.cs Out/Assembly-CSharp/ShipAlarmCord.cs
--- Temp/PatchSource/Assembly-CSharp/ShipAlarmCord.cs	2024-09-10 05:42:03.350153793 +0000
+++ Out/Assembly-CSharp/ShipAlarmCord.cs	2024-09-10 05:40:43.242150974 +0000
@@ -103,7 +103,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -146,7 +146,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/ShipBuildModeManager.cs Out/Assembly-CSharp/ShipBuildModeManager.cs
--- Temp/PatchSource/Assembly-CSharp/ShipBuildModeManager.cs	2024-09-10 05:42:03.278153790 +0000
+++ Out/Assembly-CSharp/ShipBuildModeManager.cs	2024-09-10 05:40:43.538150984 +0000
@@ -217,7 +217,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -260,7 +260,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/ShipLights.cs Out/Assembly-CSharp/ShipLights.cs
--- Temp/PatchSource/Assembly-CSharp/ShipLights.cs	2024-09-10 05:42:03.318153792 +0000
+++ Out/Assembly-CSharp/ShipLights.cs	2024-09-10 05:40:43.326150977 +0000
@@ -58,7 +58,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -87,7 +87,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/ShipTeleporter.cs Out/Assembly-CSharp/ShipTeleporter.cs
--- Temp/PatchSource/Assembly-CSharp/ShipTeleporter.cs	2024-09-10 05:42:03.342153793 +0000
+++ Out/Assembly-CSharp/ShipTeleporter.cs	2024-09-10 05:40:43.326150977 +0000
@@ -435,7 +435,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -492,7 +492,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/ShotgunItem.cs Out/Assembly-CSharp/ShotgunItem.cs
--- Temp/PatchSource/Assembly-CSharp/ShotgunItem.cs	2024-09-10 05:42:03.270153790 +0000
+++ Out/Assembly-CSharp/ShotgunItem.cs	2024-09-10 05:40:43.538150984 +0000
@@ -376,7 +376,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -419,7 +419,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/Shovel.cs Out/Assembly-CSharp/Shovel.cs
--- Temp/PatchSource/Assembly-CSharp/Shovel.cs	2024-09-10 05:42:03.286153791 +0000
+++ Out/Assembly-CSharp/Shovel.cs	2024-09-10 05:40:43.538150984 +0000
@@ -183,7 +183,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -226,7 +226,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/SoccerBallProp.cs Out/Assembly-CSharp/SoccerBallProp.cs
--- Temp/PatchSource/Assembly-CSharp/SoccerBallProp.cs	2024-09-10 05:42:03.298153791 +0000
+++ Out/Assembly-CSharp/SoccerBallProp.cs	2024-09-10 05:40:43.534150984 +0000
@@ -99,7 +99,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -128,7 +128,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/SoundManager.cs Out/Assembly-CSharp/SoundManager.cs
--- Temp/PatchSource/Assembly-CSharp/SoundManager.cs	2024-09-10 05:42:03.318153792 +0000
+++ Out/Assembly-CSharp/SoundManager.cs	2024-09-10 05:40:43.330150977 +0000
@@ -464,7 +464,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -507,7 +507,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/SpikeRoofTrap.cs Out/Assembly-CSharp/SpikeRoofTrap.cs
--- Temp/PatchSource/Assembly-CSharp/SpikeRoofTrap.cs	2024-09-10 05:42:03.278153790 +0000
+++ Out/Assembly-CSharp/SpikeRoofTrap.cs	2024-09-10 05:40:43.538150984 +0000
@@ -217,7 +217,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -260,7 +260,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/SprayPaintItem.cs Out/Assembly-CSharp/SprayPaintItem.cs
--- Temp/PatchSource/Assembly-CSharp/SprayPaintItem.cs	2024-09-10 05:42:03.282153791 +0000
+++ Out/Assembly-CSharp/SprayPaintItem.cs	2024-09-10 05:40:43.538150984 +0000
@@ -292,7 +292,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -363,7 +363,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/SpringManAI.cs Out/Assembly-CSharp/SpringManAI.cs
--- Temp/PatchSource/Assembly-CSharp/SpringManAI.cs	2024-09-10 05:42:03.286153791 +0000
+++ Out/Assembly-CSharp/SpringManAI.cs	2024-09-10 05:40:43.538150984 +0000
@@ -257,7 +257,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -314,7 +314,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/StartMatchLever.cs Out/Assembly-CSharp/StartMatchLever.cs
--- Temp/PatchSource/Assembly-CSharp/StartMatchLever.cs	2024-09-10 05:42:03.342153793 +0000
+++ Out/Assembly-CSharp/StartMatchLever.cs	2024-09-10 05:40:43.326150977 +0000
@@ -114,7 +114,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -150,7 +150,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/StartOfRound.cs Out/Assembly-CSharp/StartOfRound.cs
--- Temp/PatchSource/Assembly-CSharp/StartOfRound.cs	2024-09-10 05:42:03.310153792 +0000
+++ Out/Assembly-CSharp/StartOfRound.cs	2024-09-10 05:40:43.530150984 +0000
@@ -2560,7 +2560,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -2876,7 +2876,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/SteamValveHazard.cs Out/Assembly-CSharp/SteamValveHazard.cs
--- Temp/PatchSource/Assembly-CSharp/SteamValveHazard.cs	2024-09-10 05:42:03.262153790 +0000
+++ Out/Assembly-CSharp/SteamValveHazard.cs	2024-09-10 05:38:16.046145795 +0000
@@ -110,7 +110,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -139,7 +139,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/StoryLog.cs Out/Assembly-CSharp/StoryLog.cs
--- Temp/PatchSource/Assembly-CSharp/StoryLog.cs	2024-09-10 05:42:03.278153790 +0000
+++ Out/Assembly-CSharp/StoryLog.cs	2024-09-10 05:38:02.758145327 +0000
@@ -46,14 +46,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/StunGrenadeItem.cs Out/Assembly-CSharp/StunGrenadeItem.cs
--- Temp/PatchSource/Assembly-CSharp/StunGrenadeItem.cs	2024-09-10 05:42:03.342153793 +0000
+++ Out/Assembly-CSharp/StunGrenadeItem.cs	2024-09-10 05:40:43.326150977 +0000
@@ -239,7 +239,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -268,7 +268,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/SyncWithServerOnSpawn.cs Out/Assembly-CSharp/SyncWithServerOnSpawn.cs
--- Temp/PatchSource/Assembly-CSharp/SyncWithServerOnSpawn.cs	2024-09-10 05:42:03.326153792 +0000
+++ Out/Assembly-CSharp/SyncWithServerOnSpawn.cs	2024-09-10 05:37:54.214145026 +0000
@@ -33,14 +33,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/TVScript.cs Out/Assembly-CSharp/TVScript.cs
--- Temp/PatchSource/Assembly-CSharp/TVScript.cs	2024-09-10 05:42:03.266153790 +0000
+++ Out/Assembly-CSharp/TVScript.cs	2024-09-10 05:40:43.538150984 +0000
@@ -156,7 +156,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -220,7 +220,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/Terminal.cs Out/Assembly-CSharp/Terminal.cs
--- Temp/PatchSource/Assembly-CSharp/Terminal.cs	2024-09-10 05:42:03.318153792 +0000
+++ Out/Assembly-CSharp/Terminal.cs	2024-09-10 05:40:43.330150977 +0000
@@ -1014,7 +1014,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -1113,7 +1113,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/TerminalAccessibleObject.cs Out/Assembly-CSharp/TerminalAccessibleObject.cs
--- Temp/PatchSource/Assembly-CSharp/TerminalAccessibleObject.cs	2024-09-10 05:42:03.266153790 +0000
+++ Out/Assembly-CSharp/TerminalAccessibleObject.cs	2024-09-10 05:40:43.170150972 +0000
@@ -233,7 +233,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -262,7 +262,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/TestEnemy.cs Out/Assembly-CSharp/TestEnemy.cs
--- Temp/PatchSource/Assembly-CSharp/TestEnemy.cs	2024-09-10 05:42:03.330153792 +0000
+++ Out/Assembly-CSharp/TestEnemy.cs	2024-09-10 05:40:43.326150977 +0000
@@ -49,14 +49,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/TetraChemicalItem.cs Out/Assembly-CSharp/TetraChemicalItem.cs
--- Temp/PatchSource/Assembly-CSharp/TetraChemicalItem.cs	2024-09-10 05:42:03.278153790 +0000
+++ Out/Assembly-CSharp/TetraChemicalItem.cs	2024-09-10 05:40:43.538150984 +0000
@@ -159,7 +159,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -188,7 +188,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/TimeOfDay.cs Out/Assembly-CSharp/TimeOfDay.cs
--- Temp/PatchSource/Assembly-CSharp/TimeOfDay.cs	2024-09-10 05:42:03.290153791 +0000
+++ Out/Assembly-CSharp/TimeOfDay.cs	2024-09-10 05:40:43.538150984 +0000
@@ -535,7 +535,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -599,7 +599,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/Turret.cs Out/Assembly-CSharp/Turret.cs
--- Temp/PatchSource/Assembly-CSharp/Turret.cs	2024-09-10 05:42:03.286153791 +0000
+++ Out/Assembly-CSharp/Turret.cs	2024-09-10 05:40:43.538150984 +0000
@@ -331,7 +331,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -402,7 +402,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/UnlockableSuit.cs Out/Assembly-CSharp/UnlockableSuit.cs
--- Temp/PatchSource/Assembly-CSharp/UnlockableSuit.cs	2024-09-10 05:42:03.314153792 +0000
+++ Out/Assembly-CSharp/UnlockableSuit.cs	2024-09-10 05:40:43.330150977 +0000
@@ -70,7 +70,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -99,7 +99,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/VehicleController.cs Out/Assembly-CSharp/VehicleController.cs
--- Temp/PatchSource/Assembly-CSharp/VehicleController.cs	2024-09-10 05:42:03.310153792 +0000
+++ Out/Assembly-CSharp/VehicleController.cs	2024-09-10 05:40:43.530150984 +0000
@@ -1598,7 +1598,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -2026,7 +2026,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/WalkieTalkie.cs Out/Assembly-CSharp/WalkieTalkie.cs
--- Temp/PatchSource/Assembly-CSharp/WalkieTalkie.cs	2024-09-10 05:42:03.354153793 +0000
+++ Out/Assembly-CSharp/WalkieTalkie.cs	2024-09-10 05:40:43.242150974 +0000
@@ -375,7 +375,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
@@ -432,7 +432,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
diff -Naur -x '*.csproj' -x obj -x bin Temp/PatchSource/Assembly-CSharp/WhoopieCushionItem.cs Out/Assembly-CSharp/WhoopieCushionItem.cs
--- Temp/PatchSource/Assembly-CSharp/WhoopieCushionItem.cs	2024-09-10 05:42:03.298153791 +0000
+++ Out/Assembly-CSharp/WhoopieCushionItem.cs	2024-09-10 05:40:43.534150984 +0000
@@ -40,14 +40,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void __initializeVariables()
+	protected override void __initializeVariables()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.FamORAssem)]
-	public override string __getTypeName()
+	protected override string __getTypeName()
 	{
 		throw null;
 	}
