diff -Naur a/Assembly-CSharp-firstpass/AssetReferenceDistanceField.cs Out/Assembly-CSharp-firstpass/AssetReferenceDistanceField.cs
--- a/Assembly-CSharp-firstpass/AssetReferenceDistanceField.cs	2024-05-15 03:35:55.061273717 +0000
+++ Out/Assembly-CSharp-firstpass/AssetReferenceDistanceField.cs	2024-05-15 03:34:40.833394165 +0000
@@ -6,7 +6,7 @@
 public class AssetReferenceDistanceField : AssetReferenceT<DistanceField>
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
-	public AssetReferenceDistanceField(string guid)
+	public AssetReferenceDistanceField(string guid): base(guid)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp-firstpass/DemotingLogHandler.cs Out/Assembly-CSharp-firstpass/DemotingLogHandler.cs
--- a/Assembly-CSharp-firstpass/DemotingLogHandler.cs	2024-05-15 03:35:55.261273393 +0000
+++ Out/Assembly-CSharp-firstpass/DemotingLogHandler.cs	2024-05-15 03:34:44.793387739 +0000
@@ -5,7 +5,7 @@
 public class DemotingLogHandler : InterceptingLogHandler
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
-	public DemotingLogHandler(ILogger logger)
+	public DemotingLogHandler(ILogger logger): base(logger)
 	{
 		throw null;
 	}
