diff -Naur a/Assembly-CSharp-firstpass/Assembly-CSharp-firstpass.csproj Out/Assembly-CSharp-firstpass/Assembly-CSharp-firstpass.csproj
--- a/Assembly-CSharp-firstpass/Assembly-CSharp-firstpass.csproj	2024-05-14 22:03:50.820053739 +0000
+++ Out/Assembly-CSharp-firstpass/Assembly-CSharp-firstpass.csproj	2024-05-14 22:02:49.628052743 +0000
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AssemblyName>Assembly-CSharp-firstpass</AssemblyName>
     <GenerateAssemblyInfo>False</GenerateAssemblyInfo>
-    <TargetFramework>net40</TargetFramework>
+    <TargetFramework>net472</TargetFramework>
   </PropertyGroup>
   <PropertyGroup>
     <LangVersion>11.0</LangVersion>
