diff -Naur a/Assembly-CSharp/ActivateNextSharingJobPackage.cs Out/Assembly-CSharp/ActivateNextSharingJobPackage.cs
--- a/Assembly-CSharp/ActivateNextSharingJobPackage.cs	2024-05-09 19:26:42.796632925 +0000
+++ Out/Assembly-CSharp/ActivateNextSharingJobPackage.cs	2024-05-09 19:21:02.504622434 +0000
@@ -11,7 +11,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void SerializeData(BinarySerializer binarySerializer)
+	protected override void SerializeData(BinarySerializer binarySerializer)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/AddClipToShareQueuePackage.cs Out/Assembly-CSharp/AddClipToShareQueuePackage.cs
--- a/Assembly-CSharp/AddClipToShareQueuePackage.cs	2024-05-09 19:26:42.808632926 +0000
+++ Out/Assembly-CSharp/AddClipToShareQueuePackage.cs	2024-05-09 19:19:07.188618878 +0000
@@ -17,7 +17,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void SerializeData(BinarySerializer binarySerializer)
+	protected override void SerializeData(BinarySerializer binarySerializer)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/AmbientOcclusionSetting.cs Out/Assembly-CSharp/AmbientOcclusionSetting.cs
--- a/Assembly-CSharp/AmbientOcclusionSetting.cs	2024-05-09 19:26:43.504632947 +0000
+++ Out/Assembly-CSharp/AmbientOcclusionSetting.cs	2024-05-09 19:22:08.988624483 +0000
@@ -14,7 +14,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override int GetDefaultValue()
+	protected override int GetDefaultValue()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/BrightnessSetting.cs Out/Assembly-CSharp/BrightnessSetting.cs
--- a/Assembly-CSharp/BrightnessSetting.cs	2024-05-09 19:26:43.516632948 +0000
+++ Out/Assembly-CSharp/BrightnessSetting.cs	2024-05-09 19:18:31.064617765 +0000
@@ -18,14 +18,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override float GetDefaultValue()
+	protected override float GetDefaultValue()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override float2 GetMinMaxValue()
+	protected override float2 GetMinMaxValue()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/BudgetCost.cs Out/Assembly-CSharp/BudgetCost.cs
--- a/Assembly-CSharp/BudgetCost.cs	2024-05-09 19:26:44.820632988 +0000
+++ Out/Assembly-CSharp/BudgetCost.cs	2024-05-09 19:18:18.320617372 +0000
@@ -33,11 +33,4 @@
 	{
 		throw null;
 	}
-
-	[MethodImpl(MethodImplOptions.NoInlining)]
-	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Private)]
-	GameObject IBudgetCost.get_gameObject()
-	{
-		throw null;
-	}
 }
diff -Naur a/Assembly-CSharp/CW_TAB.cs Out/Assembly-CSharp/CW_TAB.cs
--- a/Assembly-CSharp/CW_TAB.cs	2024-05-09 19:26:43.868632958 +0000
+++ Out/Assembly-CSharp/CW_TAB.cs	2024-05-09 19:17:21.180615610 +0000
@@ -14,7 +14,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void UpdateSelection()
+	protected override void UpdateSelection()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ChromaticAberrationSetting.cs Out/Assembly-CSharp/ChromaticAberrationSetting.cs
--- a/Assembly-CSharp/ChromaticAberrationSetting.cs	2024-05-09 19:26:43.524632948 +0000
+++ Out/Assembly-CSharp/ChromaticAberrationSetting.cs	2024-05-09 19:19:03.724618772 +0000
@@ -14,7 +14,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override int GetDefaultValue()
+	protected override int GetDefaultValue()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ConnectionStateHandler.cs Out/Assembly-CSharp/ConnectionStateHandler.cs
--- a/Assembly-CSharp/ConnectionStateHandler.cs	2024-05-09 19:26:42.596632919 +0000
+++ Out/Assembly-CSharp/ConnectionStateHandler.cs	2024-05-09 19:18:01.676616858 +0000
@@ -8,7 +8,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnCreated()
+	protected override void OnCreated()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/CrouchKeybindSetting.cs Out/Assembly-CSharp/CrouchKeybindSetting.cs
--- a/Assembly-CSharp/CrouchKeybindSetting.cs	2024-05-09 19:26:43.452632946 +0000
+++ Out/Assembly-CSharp/CrouchKeybindSetting.cs	2024-05-09 19:18:12.660617197 +0000
@@ -8,7 +8,7 @@
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override KeyCode GetDefaultKey()
+	protected override KeyCode GetDefaultKey()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/CurvedUI/CurvedUIEventSystem.cs Out/Assembly-CSharp/CurvedUI/CurvedUIEventSystem.cs
--- a/Assembly-CSharp/CurvedUI/CurvedUIEventSystem.cs	2024-05-09 19:26:45.828633019 +0000
+++ Out/Assembly-CSharp/CurvedUI/CurvedUIEventSystem.cs	2024-05-09 19:18:23.512617532 +0000
@@ -11,14 +11,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void Awake()
+	protected override void Awake()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnApplicationFocus(bool hasFocus)
+	protected override void OnApplicationFocus(bool hasFocus)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/CurvedUI/CurvedUIPointerEventData.cs Out/Assembly-CSharp/CurvedUI/CurvedUIPointerEventData.cs
--- a/Assembly-CSharp/CurvedUI/CurvedUIPointerEventData.cs	2024-05-09 19:26:45.920633022 +0000
+++ Out/Assembly-CSharp/CurvedUI/CurvedUIPointerEventData.cs	2024-05-09 19:24:31.992628892 +0000
@@ -17,7 +17,7 @@
 	public Vector2 TouchPadAxis;
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
-	public CurvedUIPointerEventData(EventSystem eventSystem)
+	public CurvedUIPointerEventData(EventSystem eventSystem): base(eventSystem)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/CurvedUI/CurvedUIRaycaster.cs Out/Assembly-CSharp/CurvedUI/CurvedUIRaycaster.cs
--- a/Assembly-CSharp/CurvedUI/CurvedUIRaycaster.cs	2024-05-09 19:26:45.968633023 +0000
+++ Out/Assembly-CSharp/CurvedUI/CurvedUIRaycaster.cs	2024-05-09 19:17:49.680616489 +0000
@@ -98,14 +98,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void Awake()
+	protected override void Awake()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void Start()
+	protected override void Start()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/CurvedUI/CurvedUIVertexEffect.cs Out/Assembly-CSharp/CurvedUI/CurvedUIVertexEffect.cs
--- a/Assembly-CSharp/CurvedUI/CurvedUIVertexEffect.cs	2024-05-09 19:26:46.076633026 +0000
+++ Out/Assembly-CSharp/CurvedUI/CurvedUIVertexEffect.cs	2024-05-09 19:17:37.612616117 +0000
@@ -156,28 +156,28 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void Awake()
+	protected override void Awake()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnEnable()
+	protected override void OnEnable()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void Start()
+	protected override void Start()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnDisable()
+	protected override void OnDisable()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/CurvedUIInputModule.cs Out/Assembly-CSharp/CurvedUIInputModule.cs
--- a/Assembly-CSharp/CurvedUIInputModule.cs	2024-05-09 19:26:44.300632972 +0000
+++ Out/Assembly-CSharp/CurvedUIInputModule.cs	2024-05-09 19:13:27.704608412 +0000
@@ -418,14 +418,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void Awake()
+	protected override void Awake()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void Start()
+	protected override void Start()
 	{
 		throw null;
 	}
@@ -459,14 +459,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override MouseState GetMousePointerEventData(int id)
+	protected override MouseState GetMousePointerEventData(int id)
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void ProcessDrag(PointerEventData pointerEvent)
+	protected override void ProcessDrag(PointerEventData pointerEvent)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/DefaultNamespace/_10/_SCRIPTS/NetworkStats.cs Out/Assembly-CSharp/DefaultNamespace/_10/_SCRIPTS/NetworkStats.cs
--- a/Assembly-CSharp/DefaultNamespace/_10/_SCRIPTS/NetworkStats.cs	2024-05-09 19:26:45.664633014 +0000
+++ Out/Assembly-CSharp/DefaultNamespace/_10/_SCRIPTS/NetworkStats.cs	2024-05-09 19:17:43.868616309 +0000
@@ -9,7 +9,7 @@
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnCreated()
+	protected override void OnCreated()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/DropKeybindSetting.cs Out/Assembly-CSharp/DropKeybindSetting.cs
--- a/Assembly-CSharp/DropKeybindSetting.cs	2024-05-09 19:26:43.460632946 +0000
+++ Out/Assembly-CSharp/DropKeybindSetting.cs	2024-05-09 19:20:36.640621636 +0000
@@ -8,7 +8,7 @@
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override KeyCode GetDefaultKey()
+	protected override KeyCode GetDefaultKey()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/EdgeDetectionSetting.cs Out/Assembly-CSharp/EdgeDetectionSetting.cs
--- a/Assembly-CSharp/EdgeDetectionSetting.cs	2024-05-09 19:26:43.540632948 +0000
+++ Out/Assembly-CSharp/EdgeDetectionSetting.cs	2024-05-09 19:19:24.372619408 +0000
@@ -14,7 +14,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override int GetDefaultValue()
+	protected override int GetDefaultValue()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/EmoteKeybindSetting.cs Out/Assembly-CSharp/EmoteKeybindSetting.cs
--- a/Assembly-CSharp/EmoteKeybindSetting.cs	2024-05-09 19:26:43.468632946 +0000
+++ Out/Assembly-CSharp/EmoteKeybindSetting.cs	2024-05-09 19:19:11.476619011 +0000
@@ -8,7 +8,7 @@
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override KeyCode GetDefaultKey()
+	protected override KeyCode GetDefaultKey()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ExtractMachineCheckItemState.cs Out/Assembly-CSharp/ExtractMachineCheckItemState.cs
--- a/Assembly-CSharp/ExtractMachineCheckItemState.cs	2024-05-09 19:26:43.012632932 +0000
+++ Out/Assembly-CSharp/ExtractMachineCheckItemState.cs	2024-05-09 19:24:10.252628222 +0000
@@ -9,7 +9,7 @@
 	public Optionable<float> m_timeBeenSearchingForItem;
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
-	public ExtractMachineCheckItemState(ExtractVideoMachine machine)
+	public ExtractMachineCheckItemState(ExtractVideoMachine machine): base(machine)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ExtractMachineClosedState.cs Out/Assembly-CSharp/ExtractMachineClosedState.cs
--- a/Assembly-CSharp/ExtractMachineClosedState.cs	2024-05-09 19:26:43.016632932 +0000
+++ Out/Assembly-CSharp/ExtractMachineClosedState.cs	2024-05-09 19:24:12.728628299 +0000
@@ -3,7 +3,7 @@
 public class ExtractMachineClosedState : VideoExtractMachineState
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
-	public ExtractMachineClosedState(ExtractVideoMachine machine)
+	public ExtractMachineClosedState(ExtractVideoMachine machine): base(machine)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ExtractMachineEjectState.cs Out/Assembly-CSharp/ExtractMachineEjectState.cs
--- a/Assembly-CSharp/ExtractMachineEjectState.cs	2024-05-09 19:26:42.864632927 +0000
+++ Out/Assembly-CSharp/ExtractMachineEjectState.cs	2024-05-09 19:24:15.940628398 +0000
@@ -3,7 +3,7 @@
 public class ExtractMachineEjectState : VideoExtractMachineState
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
-	public ExtractMachineEjectState(ExtractVideoMachine machine)
+	public ExtractMachineEjectState(ExtractVideoMachine machine): base(machine)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ExtractMachineExtractingState.cs Out/Assembly-CSharp/ExtractMachineExtractingState.cs
--- a/Assembly-CSharp/ExtractMachineExtractingState.cs	2024-05-09 19:26:42.868632928 +0000
+++ Out/Assembly-CSharp/ExtractMachineExtractingState.cs	2024-05-09 19:24:06.732628114 +0000
@@ -3,7 +3,7 @@
 public class ExtractMachineExtractingState : VideoExtractMachineState
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
-	public ExtractMachineExtractingState(ExtractVideoMachine machine)
+	public ExtractMachineExtractingState(ExtractVideoMachine machine): base(machine)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ExtractMachineIdleState.cs Out/Assembly-CSharp/ExtractMachineIdleState.cs
--- a/Assembly-CSharp/ExtractMachineIdleState.cs	2024-05-09 19:26:42.876632928 +0000
+++ Out/Assembly-CSharp/ExtractMachineIdleState.cs	2024-05-09 19:24:00.608627925 +0000
@@ -3,7 +3,7 @@
 public class ExtractMachineIdleState : VideoExtractMachineState
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
-	public ExtractMachineIdleState(ExtractVideoMachine machine)
+	public ExtractMachineIdleState(ExtractVideoMachine machine): base(machine)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/FullscreenSetting.cs Out/Assembly-CSharp/FullscreenSetting.cs
--- a/Assembly-CSharp/FullscreenSetting.cs	2024-05-09 19:26:43.548632949 +0000
+++ Out/Assembly-CSharp/FullscreenSetting.cs	2024-05-09 19:18:08.552617070 +0000
@@ -14,7 +14,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override int GetDefaultValue()
+	protected override int GetDefaultValue()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/GameLoopHax.cs Out/Assembly-CSharp/GameLoopHax.cs
--- a/Assembly-CSharp/GameLoopHax.cs	2024-05-09 19:26:43.020632932 +0000
+++ Out/Assembly-CSharp/GameLoopHax.cs	2024-05-09 19:18:59.536618642 +0000
@@ -259,7 +259,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnCreated()
+	protected override void OnCreated()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/GlobalInputHandler.cs Out/Assembly-CSharp/GlobalInputHandler.cs
--- a/Assembly-CSharp/GlobalInputHandler.cs	2024-05-09 19:26:43.040632933 +0000
+++ Out/Assembly-CSharp/GlobalInputHandler.cs	2024-05-09 19:13:30.396608495 +0000
@@ -67,7 +67,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnCreated()
+	protected override void OnCreated()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/InfiltratorSpawner.cs Out/Assembly-CSharp/InfiltratorSpawner.cs
--- a/Assembly-CSharp/InfiltratorSpawner.cs	2024-05-09 19:26:44.556632980 +0000
+++ Out/Assembly-CSharp/InfiltratorSpawner.cs	2024-05-09 19:16:32.188614099 +0000
@@ -78,11 +78,4 @@
 	{
 		throw null;
 	}
-
-	[MethodImpl(MethodImplOptions.NoInlining)]
-	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Private)]
-	GameObject IBudgetCost.get_gameObject()
-	{
-		throw null;
-	}
 }
diff -Naur a/Assembly-CSharp/InteractKeybindSetting.cs Out/Assembly-CSharp/InteractKeybindSetting.cs
--- a/Assembly-CSharp/InteractKeybindSetting.cs	2024-05-09 19:26:43.476632946 +0000
+++ Out/Assembly-CSharp/InteractKeybindSetting.cs	2024-05-09 19:18:05.004616961 +0000
@@ -8,7 +8,7 @@
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override KeyCode GetDefaultKey()
+	protected override KeyCode GetDefaultKey()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/InvertedMouseSetting.cs Out/Assembly-CSharp/InvertedMouseSetting.cs
--- a/Assembly-CSharp/InvertedMouseSetting.cs	2024-05-09 19:26:43.560632949 +0000
+++ Out/Assembly-CSharp/InvertedMouseSetting.cs	2024-05-09 19:16:56.388614846 +0000
@@ -14,7 +14,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override int GetDefaultValue()
+	protected override int GetDefaultValue()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ItemInstanceDataHandler.cs Out/Assembly-CSharp/ItemInstanceDataHandler.cs
--- a/Assembly-CSharp/ItemInstanceDataHandler.cs	2024-05-09 19:26:43.060632933 +0000
+++ Out/Assembly-CSharp/ItemInstanceDataHandler.cs	2024-05-09 19:16:37.428614261 +0000
@@ -11,7 +11,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnCreated()
+	protected override void OnCreated()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ItemInstanceHandler.cs Out/Assembly-CSharp/ItemInstanceHandler.cs
--- a/Assembly-CSharp/ItemInstanceHandler.cs	2024-05-09 19:26:43.076632934 +0000
+++ Out/Assembly-CSharp/ItemInstanceHandler.cs	2024-05-09 19:15:26.676612080 +0000
@@ -19,7 +19,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnCreated()
+	protected override void OnCreated()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ItemInstancePackage.cs Out/Assembly-CSharp/ItemInstancePackage.cs
--- a/Assembly-CSharp/ItemInstancePackage.cs	2024-05-09 19:26:42.816632926 +0000
+++ Out/Assembly-CSharp/ItemInstancePackage.cs	2024-05-09 19:14:54.888611100 +0000
@@ -17,7 +17,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void SerializeData(BinarySerializer binarySerializer)
+	protected override void SerializeData(BinarySerializer binarySerializer)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/JumpKeybindSetting.cs Out/Assembly-CSharp/JumpKeybindSetting.cs
--- a/Assembly-CSharp/JumpKeybindSetting.cs	2024-05-09 19:26:43.484632947 +0000
+++ Out/Assembly-CSharp/JumpKeybindSetting.cs	2024-05-09 19:16:40.972614370 +0000
@@ -14,7 +14,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override KeyCode GetDefaultKey()
+	protected override KeyCode GetDefaultKey()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/MainMenuUIHandler.cs Out/Assembly-CSharp/MainMenuUIHandler.cs
--- a/Assembly-CSharp/MainMenuUIHandler.cs	2024-05-09 19:26:43.656632952 +0000
+++ Out/Assembly-CSharp/MainMenuUIHandler.cs	2024-05-09 19:20:32.056621495 +0000
@@ -7,7 +7,7 @@
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnTransistionedToPage(UIPage newPage)
+	protected override void OnTransistionedToPage(UIPage newPage)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/MasterVolumeSetting.cs Out/Assembly-CSharp/MasterVolumeSetting.cs
--- a/Assembly-CSharp/MasterVolumeSetting.cs	2024-05-09 19:26:43.548632949 +0000
+++ Out/Assembly-CSharp/MasterVolumeSetting.cs	2024-05-09 19:20:24.384621258 +0000
@@ -17,14 +17,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override float GetDefaultValue()
+	protected override float GetDefaultValue()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override float2 GetMinMaxValue()
+	protected override float2 GetMinMaxValue()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/MaxFramerateSetting.cs Out/Assembly-CSharp/MaxFramerateSetting.cs
--- a/Assembly-CSharp/MaxFramerateSetting.cs	2024-05-09 19:26:43.556632949 +0000
+++ Out/Assembly-CSharp/MaxFramerateSetting.cs	2024-05-09 19:20:18.824621087 +0000
@@ -14,14 +14,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override float GetDefaultValue()
+	protected override float GetDefaultValue()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override float2 GetMinMaxValue()
+	protected override float2 GetMinMaxValue()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/MetaProgressionHandler.cs Out/Assembly-CSharp/MetaProgressionHandler.cs
--- a/Assembly-CSharp/MetaProgressionHandler.cs	2024-05-09 19:26:45.112632997 +0000
+++ Out/Assembly-CSharp/MetaProgressionHandler.cs	2024-05-09 19:19:20.200619279 +0000
@@ -26,7 +26,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnCreated()
+	protected override void OnCreated()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/Modal.cs Out/Assembly-CSharp/Modal.cs
--- a/Assembly-CSharp/Modal.cs	2024-05-09 19:26:43.236632939 +0000
+++ Out/Assembly-CSharp/Modal.cs	2024-05-09 19:18:38.872618005 +0000
@@ -33,7 +33,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnCreated()
+	protected override void OnCreated()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/MonsterGroupClose.cs Out/Assembly-CSharp/MonsterGroupClose.cs
--- a/Assembly-CSharp/MonsterGroupClose.cs	2024-05-09 19:26:45.212633000 +0000
+++ Out/Assembly-CSharp/MonsterGroupClose.cs	2024-05-09 19:17:57.172616720 +0000
@@ -159,11 +159,4 @@
 	{
 		throw null;
 	}
-
-	[MethodImpl(MethodImplOptions.NoInlining)]
-	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Private)]
-	GameObject IBudgetCost.get_gameObject()
-	{
-		throw null;
-	}
 }
diff -Naur a/Assembly-CSharp/MouseSensitivitySetting.cs Out/Assembly-CSharp/MouseSensitivitySetting.cs
--- a/Assembly-CSharp/MouseSensitivitySetting.cs	2024-05-09 19:26:43.568632949 +0000
+++ Out/Assembly-CSharp/MouseSensitivitySetting.cs	2024-05-09 19:18:54.208618478 +0000
@@ -14,14 +14,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override float GetDefaultValue()
+	protected override float GetDefaultValue()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override float2 GetMinMaxValue()
+	protected override float2 GetMinMaxValue()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/PersistentObjectsHolder.cs Out/Assembly-CSharp/PersistentObjectsHolder.cs
--- a/Assembly-CSharp/PersistentObjectsHolder.cs	2024-05-09 19:26:43.400632944 +0000
+++ Out/Assembly-CSharp/PersistentObjectsHolder.cs	2024-05-09 19:14:32.408610406 +0000
@@ -91,7 +91,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnCreated()
+	protected override void OnCreated()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/PhotonSendVideoHandler.cs Out/Assembly-CSharp/PhotonSendVideoHandler.cs
--- a/Assembly-CSharp/PhotonSendVideoHandler.cs	2024-05-09 19:26:43.184632937 +0000
+++ Out/Assembly-CSharp/PhotonSendVideoHandler.cs	2024-05-09 19:15:46.112612679 +0000
@@ -182,7 +182,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnCreated()
+	protected override void OnCreated()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/PushToTalkButtonSetting.cs Out/Assembly-CSharp/PushToTalkButtonSetting.cs
--- a/Assembly-CSharp/PushToTalkButtonSetting.cs	2024-05-09 19:26:43.492632947 +0000
+++ Out/Assembly-CSharp/PushToTalkButtonSetting.cs	2024-05-09 19:14:28.596610289 +0000
@@ -8,7 +8,7 @@
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override KeyCode GetDefaultKey()
+	protected override KeyCode GetDefaultKey()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ReRequestClipPackage.cs Out/Assembly-CSharp/ReRequestClipPackage.cs
--- a/Assembly-CSharp/ReRequestClipPackage.cs	2024-05-09 19:26:42.828632926 +0000
+++ Out/Assembly-CSharp/ReRequestClipPackage.cs	2024-05-09 19:14:36.148610522 +0000
@@ -13,7 +13,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void SerializeData(BinarySerializer binarySerializer)
+	protected override void SerializeData(BinarySerializer binarySerializer)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/RecordingsHandler.cs Out/Assembly-CSharp/RecordingsHandler.cs
--- a/Assembly-CSharp/RecordingsHandler.cs	2024-05-09 19:26:43.764632955 +0000
+++ Out/Assembly-CSharp/RecordingsHandler.cs	2024-05-09 19:15:37.512612414 +0000
@@ -252,7 +252,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnCreated()
+	protected override void OnCreated()
 	{
 		throw null;
 	}
@@ -459,7 +459,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnRemoved()
+	protected override void OnRemoved()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/RenderScaleSetting.cs Out/Assembly-CSharp/RenderScaleSetting.cs
--- a/Assembly-CSharp/RenderScaleSetting.cs	2024-05-09 19:26:43.504632947 +0000
+++ Out/Assembly-CSharp/RenderScaleSetting.cs	2024-05-09 19:14:50.240610956 +0000
@@ -14,14 +14,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override float GetDefaultValue()
+	protected override float GetDefaultValue()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override float2 GetMinMaxValue()
+	protected override float2 GetMinMaxValue()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/SFXVolumeSetting.cs Out/Assembly-CSharp/SFXVolumeSetting.cs
--- a/Assembly-CSharp/SFXVolumeSetting.cs	2024-05-09 19:26:43.532632948 +0000
+++ Out/Assembly-CSharp/SFXVolumeSetting.cs	2024-05-09 19:20:41.864621797 +0000
@@ -17,14 +17,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override float GetDefaultValue()
+	protected override float GetDefaultValue()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override float2 GetMinMaxValue()
+	protected override float2 GetMinMaxValue()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/SaveUICell.cs Out/Assembly-CSharp/SaveUICell.cs
--- a/Assembly-CSharp/SaveUICell.cs	2024-05-09 19:26:43.672632952 +0000
+++ Out/Assembly-CSharp/SaveUICell.cs	2024-05-09 19:20:56.176622239 +0000
@@ -50,7 +50,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void UpdateSelection()
+	protected override void UpdateSelection()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/SaveUICellTABS.cs Out/Assembly-CSharp/SaveUICellTABS.cs
--- a/Assembly-CSharp/SaveUICellTABS.cs	2024-05-09 19:26:43.680632953 +0000
+++ Out/Assembly-CSharp/SaveUICellTABS.cs	2024-05-09 19:20:52.476622124 +0000
@@ -7,7 +7,7 @@
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void Start()
+	protected override void Start()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/SendClipCompletedPackage.cs Out/Assembly-CSharp/SendClipCompletedPackage.cs
--- a/Assembly-CSharp/SendClipCompletedPackage.cs	2024-05-09 19:26:42.836632927 +0000
+++ Out/Assembly-CSharp/SendClipCompletedPackage.cs	2024-05-09 19:20:48.980622017 +0000
@@ -13,7 +13,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void SerializeData(BinarySerializer binarySerializer)
+	protected override void SerializeData(BinarySerializer binarySerializer)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/SendVideoChunkPackage.cs Out/Assembly-CSharp/SendVideoChunkPackage.cs
--- a/Assembly-CSharp/SendVideoChunkPackage.cs	2024-05-09 19:26:43.080632934 +0000
+++ Out/Assembly-CSharp/SendVideoChunkPackage.cs	2024-05-09 19:19:28.404619532 +0000
@@ -22,7 +22,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void SerializeData(BinarySerializer binarySerializer)
+	protected override void SerializeData(BinarySerializer binarySerializer)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ShadowQualitySetting.cs Out/Assembly-CSharp/ShadowQualitySetting.cs
--- a/Assembly-CSharp/ShadowQualitySetting.cs	2024-05-09 19:26:43.544632948 +0000
+++ Out/Assembly-CSharp/ShadowQualitySetting.cs	2024-05-09 19:19:14.552619105 +0000
@@ -22,7 +22,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override int GetDefaultValue()
+	protected override int GetDefaultValue()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/SprintKeybindSetting.cs Out/Assembly-CSharp/SprintKeybindSetting.cs
--- a/Assembly-CSharp/SprintKeybindSetting.cs	2024-05-09 19:26:43.496632947 +0000
+++ Out/Assembly-CSharp/SprintKeybindSetting.cs	2024-05-09 19:16:52.116614714 +0000
@@ -8,7 +8,7 @@
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override KeyCode GetDefaultKey()
+	protected override KeyCode GetDefaultKey()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/StartRecordingCommandPackage.cs Out/Assembly-CSharp/StartRecordingCommandPackage.cs
--- a/Assembly-CSharp/StartRecordingCommandPackage.cs	2024-05-09 19:26:42.848632927 +0000
+++ Out/Assembly-CSharp/StartRecordingCommandPackage.cs	2024-05-09 19:16:49.132614622 +0000
@@ -18,7 +18,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void SerializeData(BinarySerializer binarySerializer)
+	protected override void SerializeData(BinarySerializer binarySerializer)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/SteamAvatarHandler.cs Out/Assembly-CSharp/SteamAvatarHandler.cs
--- a/Assembly-CSharp/SteamAvatarHandler.cs	2024-05-09 19:26:43.780632956 +0000
+++ Out/Assembly-CSharp/SteamAvatarHandler.cs	2024-05-09 19:18:48.364618298 +0000
@@ -16,7 +16,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnCreated()
+	protected override void OnCreated()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/StopRecordingCommandPackage.cs Out/Assembly-CSharp/StopRecordingCommandPackage.cs
--- a/Assembly-CSharp/StopRecordingCommandPackage.cs	2024-05-09 19:26:42.856632927 +0000
+++ Out/Assembly-CSharp/StopRecordingCommandPackage.cs	2024-05-09 19:16:45.300614504 +0000
@@ -14,7 +14,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void SerializeData(BinarySerializer binarySerializer)
+	protected override void SerializeData(BinarySerializer binarySerializer)
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/TimeOfDayHandler.cs Out/Assembly-CSharp/TimeOfDayHandler.cs
--- a/Assembly-CSharp/TimeOfDayHandler.cs	2024-05-09 19:26:43.796632956 +0000
+++ Out/Assembly-CSharp/TimeOfDayHandler.cs	2024-05-09 19:15:30.232612189 +0000
@@ -17,7 +17,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnCreated()
+	protected override void OnCreated()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/ToggleSelfieModeKeybindSetting.cs Out/Assembly-CSharp/ToggleSelfieModeKeybindSetting.cs
--- a/Assembly-CSharp/ToggleSelfieModeKeybindSetting.cs	2024-05-09 19:26:43.504632947 +0000
+++ Out/Assembly-CSharp/ToggleSelfieModeKeybindSetting.cs	2024-05-09 19:15:22.660611956 +0000
@@ -8,7 +8,7 @@
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override KeyCode GetDefaultKey()
+	protected override KeyCode GetDefaultKey()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/TransitionHandler.cs Out/Assembly-CSharp/TransitionHandler.cs
--- a/Assembly-CSharp/TransitionHandler.cs	2024-05-09 19:26:43.824632957 +0000
+++ Out/Assembly-CSharp/TransitionHandler.cs	2024-05-09 19:14:59.948611256 +0000
@@ -23,7 +23,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnCreated()
+	protected override void OnCreated()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/UnityEngine/UI/ProceduralImage/ProceduralImage.cs Out/Assembly-CSharp/UnityEngine/UI/ProceduralImage/ProceduralImage.cs
--- a/Assembly-CSharp/UnityEngine/UI/ProceduralImage/ProceduralImage.cs	2024-05-09 19:26:46.180633030 +0000
+++ Out/Assembly-CSharp/UnityEngine/UI/ProceduralImage/ProceduralImage.cs	2024-05-09 19:13:44.260608922 +0000
@@ -113,14 +113,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnEnable()
+	protected override void OnEnable()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnDisable()
+	protected override void OnDisable()
 	{
 		throw null;
 	}
@@ -162,14 +162,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnPopulateMesh(VertexHelper toFill)
+	protected override void OnPopulateMesh(VertexHelper toFill)
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override void OnTransformParentChanged()
+	protected override void OnTransformParentChanged()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/VSyncSetting.cs Out/Assembly-CSharp/VSyncSetting.cs
--- a/Assembly-CSharp/VSyncSetting.cs	2024-05-09 19:26:43.628632951 +0000
+++ Out/Assembly-CSharp/VSyncSetting.cs	2024-05-09 19:13:51.484609145 +0000
@@ -14,7 +14,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override int GetDefaultValue()
+	protected override int GetDefaultValue()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/VoiceChatModeSetting.cs Out/Assembly-CSharp/VoiceChatModeSetting.cs
--- a/Assembly-CSharp/VoiceChatModeSetting.cs	2024-05-09 19:26:43.580632950 +0000
+++ Out/Assembly-CSharp/VoiceChatModeSetting.cs	2024-05-09 19:14:21.928610083 +0000
@@ -14,7 +14,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override int GetDefaultValue()
+	protected override int GetDefaultValue()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/VoiceVolumeSetting.cs Out/Assembly-CSharp/VoiceVolumeSetting.cs
--- a/Assembly-CSharp/VoiceVolumeSetting.cs	2024-05-09 19:26:43.616632951 +0000
+++ Out/Assembly-CSharp/VoiceVolumeSetting.cs	2024-05-09 19:13:55.828609279 +0000
@@ -23,14 +23,14 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override float GetDefaultValue()
+	protected override float GetDefaultValue()
 	{
 		throw null;
 	}
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override float2 GetMinMaxValue()
+	protected override float2 GetMinMaxValue()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/WalkBackwardKeybindSetting.cs Out/Assembly-CSharp/WalkBackwardKeybindSetting.cs
--- a/Assembly-CSharp/WalkBackwardKeybindSetting.cs	2024-05-09 19:26:43.512632947 +0000
+++ Out/Assembly-CSharp/WalkBackwardKeybindSetting.cs	2024-05-09 19:14:16.048609902 +0000
@@ -8,7 +8,7 @@
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override KeyCode GetDefaultKey()
+	protected override KeyCode GetDefaultKey()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/WalkForwardKeybindSetting.cs Out/Assembly-CSharp/WalkForwardKeybindSetting.cs
--- a/Assembly-CSharp/WalkForwardKeybindSetting.cs	2024-05-09 19:26:43.520632948 +0000
+++ Out/Assembly-CSharp/WalkForwardKeybindSetting.cs	2024-05-09 19:14:25.236610185 +0000
@@ -8,7 +8,7 @@
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override KeyCode GetDefaultKey()
+	protected override KeyCode GetDefaultKey()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/WalkLeftKeybindSetting.cs Out/Assembly-CSharp/WalkLeftKeybindSetting.cs
--- a/Assembly-CSharp/WalkLeftKeybindSetting.cs	2024-05-09 19:26:43.528632948 +0000
+++ Out/Assembly-CSharp/WalkLeftKeybindSetting.cs	2024-05-09 19:14:09.628609704 +0000
@@ -8,7 +8,7 @@
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override KeyCode GetDefaultKey()
+	protected override KeyCode GetDefaultKey()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/WalkRightKeybindSetting.cs Out/Assembly-CSharp/WalkRightKeybindSetting.cs
--- a/Assembly-CSharp/WalkRightKeybindSetting.cs	2024-05-09 19:26:43.536632948 +0000
+++ Out/Assembly-CSharp/WalkRightKeybindSetting.cs	2024-05-09 19:14:05.224609568 +0000
@@ -8,7 +8,7 @@
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override KeyCode GetDefaultKey()
+	protected override KeyCode GetDefaultKey()
 	{
 		throw null;
 	}
diff -Naur a/Assembly-CSharp/WorldShaderSetting.cs Out/Assembly-CSharp/WorldShaderSetting.cs
--- a/Assembly-CSharp/WorldShaderSetting.cs	2024-05-09 19:26:43.636632951 +0000
+++ Out/Assembly-CSharp/WorldShaderSetting.cs	2024-05-09 19:14:01.524609454 +0000
@@ -22,7 +22,7 @@
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
 	[BepInEx.AssemblyPublicizer.OriginalAttributes(MethodAttributes.Family)]
-	public override int GetDefaultValue()
+	protected override int GetDefaultValue()
 	{
 		throw null;
 	}
diff -Naur a/Zorro.Core.Runtime/Zorro/Core/NativeHashedBookkeeper.cs Out/Zorro.Core.Runtime/Zorro/Core/NativeHashedBookkeeper.cs
--- a/Zorro.Core.Runtime/Zorro/Core/NativeHashedBookkeeper.cs	2024-05-09 19:26:46.360633035 +0000
+++ Out/Zorro.Core.Runtime/Zorro/Core/NativeHashedBookkeeper.cs	2024-05-09 19:25:47.112631209 +0000
@@ -8,7 +8,7 @@
 	public BidirectionalNativeDictionary<T, int> IndexHashMap;
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
-	public NativeHashedBookkeeper(int defaultCapacity)
+	public NativeHashedBookkeeper(int defaultCapacity): base(defaultCapacity)
 	{
 		throw null;
 	}
diff -Naur a/Zorro.Core.Runtime/Zorro/Core/ObjectNativeBookkeeper.cs Out/Zorro.Core.Runtime/Zorro/Core/ObjectNativeBookkeeper.cs
--- a/Zorro.Core.Runtime/Zorro/Core/ObjectNativeBookkeeper.cs	2024-05-09 19:26:46.384633036 +0000
+++ Out/Zorro.Core.Runtime/Zorro/Core/ObjectNativeBookkeeper.cs	2024-05-09 19:25:44.140631117 +0000
@@ -11,7 +11,7 @@
 	public BidirectionalNativeDictionary<int, int> InstanceIDtoIndexHash;
 
 	[MethodImpl(MethodImplOptions.NoInlining)]
-	public ObjectNativeBookkeeper(int defaultCapacity)
+	public ObjectNativeBookkeeper(int defaultCapacity): base(defaultCapacity)
 	{
 		throw null;
 	}
diff -Naur a/Zorro.Core.Runtime/__JobReflectionRegistrationOutput__3156083764.cs Out/Zorro.Core.Runtime/__JobReflectionRegistrationOutput__3156083764.cs
--- a/Zorro.Core.Runtime/__JobReflectionRegistrationOutput__3156083764.cs	2024-05-09 19:26:46.464633038 +0000
+++ Out/Zorro.Core.Runtime/__JobReflectionRegistrationOutput__3156083764.cs	2024-05-09 19:22:20.736624846 +0000
@@ -2,7 +2,6 @@
 using Unity.Jobs;
 using UnityEngine;
 
-[Unity.Jobs.DOTSCompilerGenerated]
 internal class __JobReflectionRegistrationOutput__3156083764
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
diff -Naur a/Zorro.PhotonUtility/Zorro.PhotonUtility.csproj Out/Zorro.PhotonUtility/Zorro.PhotonUtility.csproj
--- a/Zorro.PhotonUtility/Zorro.PhotonUtility.csproj	2024-05-09 19:27:02.912633546 +0000
+++ Out/Zorro.PhotonUtility/Zorro.PhotonUtility.csproj	2024-05-09 19:24:59.280629734 +0000
@@ -21,6 +21,12 @@
     <Reference Include="Zorro.Core.Runtime">
       <HintPath>/workspaces/DocumentationWarning/_tmp/assemblies/Zorro.Core.Runtime.dll</HintPath>
     </Reference>
+    <ItemGroup><Reference Include="UnityEngine">
+      <HintPath>/workspaces/DocumentationWarning/_tmp/assemblies/UnityEngine.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEngine.CoreModule">
+      <HintPath>/workspaces/DocumentationWarning/_tmp/assemblies/UnityEngine.CoreModule.dll</HintPath>
+    </Reference></ItemGroup>
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="UnityEngine.Modules" Version="2022.3.10">
diff -Naur a/Zorro.Recorder/__JobReflectionRegistrationOutput__2199089951.cs Out/Zorro.Recorder/__JobReflectionRegistrationOutput__2199089951.cs
--- a/Zorro.Recorder/__JobReflectionRegistrationOutput__2199089951.cs	2024-05-09 19:26:46.676633045 +0000
+++ Out/Zorro.Recorder/__JobReflectionRegistrationOutput__2199089951.cs	2024-05-09 19:25:51.160631333 +0000
@@ -2,7 +2,6 @@
 using Unity.Jobs;
 using UnityEngine;
 
-[Unity.Jobs.DOTSCompilerGenerated]
 internal class __JobReflectionRegistrationOutput__2199089951
 {
 	[MethodImpl(MethodImplOptions.NoInlining)]
diff -Naur a/Zorro.Settings.Runtime/Zorro.Settings.Runtime.csproj Out/Zorro.Settings.Runtime/Zorro.Settings.Runtime.csproj
--- a/Zorro.Settings.Runtime/Zorro.Settings.Runtime.csproj	2024-05-09 19:26:58.388633406 +0000
+++ Out/Zorro.Settings.Runtime/Zorro.Settings.Runtime.csproj	2024-05-09 19:25:27.080630591 +0000
@@ -30,6 +30,9 @@
     <Reference Include="Zorro.Core.Runtime">
       <HintPath>/workspaces/DocumentationWarning/_tmp/assemblies/Zorro.Core.Runtime.dll</HintPath>
     </Reference>
+    <ItemGroup><Reference Include="Sirenix.Serialization">
+      <HintPath>/workspaces/DocumentationWarning/_tmp/assemblies/Sirenix.Serialization.dll</HintPath>
+    </Reference></ItemGroup>
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="UnityEngine.Modules" Version="2022.3.10">
